Option 1 -  Crawl news & information websites & anticipate the likelihood of its virality.

In [1]:
#Using twitterscraper API for mining tweets

from twitterscraper import query_tweets
import datetime as dt
import pandas as pd

INFO: {'User-Agent': 'Opera/9.80 (X11; Linux i686; Ubuntu/14.10) Presto/2.12.388 Version/12.16'}


In [50]:
#Using begin and end dates for mining recent tweets
begin_date = dt.date(2020,3,1) 
end_date = dt.date(2020,4,21)
limit = 3000
lang = 'English'

In [35]:
#Tweets in Hyderabad area about corona or corona virus
hyd_tweets = query_tweets(begindate=begin_date, enddate=end_date,limit=limit, lang=lang, query='Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Hyderabad within:mi' )

INFO: queries: ['Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Hyderabad within:500mi since:2020-03-01 until:2020-03-03', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Hyderabad within:500mi since:2020-03-03 until:2020-03-06', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Hyderabad within:500mi since:2020-03-06 until:2020-03-08', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Hyderabad within:500mi since:2020-03-08 until:2020-03-11', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Hyderabad within:500mi since:2020-03-11 until:2020-03-13', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Hyderabad within:

In [36]:
hyd_df = pd.DataFrame(t.__dict__ for t in hyd_tweets)

In [39]:
#checking hyderabad tweets
hyd_df.describe()

timestamp_epochs          likes      retweets       replies
count      4.802000e+03    4802.000000   4802.000000   4802.000000
mean       1.585606e+09     436.023324    147.158059     22.683049
std        1.211177e+06    3056.209088   1410.168903    299.025321
min        1.583036e+09       8.000000      0.000000      0.000000
25%        1.584680e+09      50.000000     21.000000      2.000000
50%        1.585549e+09      99.500000     34.000000      5.000000
75%        1.586721e+09     239.000000     71.000000     13.000000
max        1.587427e+09  151021.000000  60354.000000  19546.000000

In [19]:
df['retweets'].describe()

count    1063.000000
mean        0.835372
std         4.554771
min         0.000000
25%         0.000000
50%         0.000000
75%         0.000000
max        71.000000
Name: retweets, dtype: float64

In [40]:
mumbai_tweets = query_tweets(begindate=begin_date, enddate=end_date,limit=limit, lang=lang, query='Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Mumbai within:200mi' )

INFO: queries: ['Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Mumbai within:200mi since:2020-03-01 until:2020-03-03', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Mumbai within:200mi since:2020-03-03 until:2020-03-06', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Mumbai within:200mi since:2020-03-06 until:2020-03-08', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Mumbai within:200mi since:2020-03-08 until:2020-03-11', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Mumbai within:200mi since:2020-03-11 until:2020-03-13', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Mumbai within:200mi since:2020-0

In [41]:
mum_df = pd.DataFrame(t.__dict__ for t in mumbai_tweets)

In [42]:
mum_df.describe()

timestamp_epochs          likes      retweets       replies
count      1.805000e+03    1805.000000   1805.000000   1805.000000
mean       1.586347e+09     568.665374    157.120222     45.874238
std        1.195982e+06    3358.374060   1164.652264    654.934983
min        1.583042e+09      10.000000      8.000000      0.000000
25%        1.585465e+09      49.000000     18.000000      2.000000
50%        1.587030e+09     108.000000     32.000000      6.000000
75%        1.587312e+09     292.000000     72.000000     17.000000
max        1.587418e+09  115470.000000  42645.000000  19546.000000

In [51]:
delhi_tweets = query_tweets(begindate=begin_date, enddate=end_date,limit=limit, lang=lang, query='Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Delhi within:200mi ' )

INFO: queries: ['Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Delhi within:200mi  since:2020-03-01 until:2020-03-03', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Delhi within:200mi  since:2020-03-03 until:2020-03-06', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Delhi within:200mi  since:2020-03-06 until:2020-03-08', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Delhi within:200mi  since:2020-03-08 until:2020-03-11', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Delhi within:200mi  since:2020-03-11 until:2020-03-13', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Delhi within:200mi  since:2020-0

In [52]:
del_df = pd.DataFrame(t.__dict__ for t in delhi_tweets)

In [53]:
del_df.describe()

timestamp_epochs          likes      retweets      replies
count      1.565000e+03    1565.000000   1565.000000  1565.000000
mean       1.585572e+09     583.790415    138.428754    25.272843
std        1.257437e+06    5548.185803    924.570151   200.578528
min        1.583039e+09      11.000000      6.000000     0.000000
25%        1.584630e+09      49.000000     21.000000     2.000000
50%        1.585558e+09      91.000000     34.000000     5.000000
75%        1.586766e+09     226.000000     73.000000    13.000000
max        1.587425e+09  151021.000000  24420.000000  5389.000000

In [54]:
banglore_tweets = query_tweets(begindate=begin_date, enddate=end_date,limit=limit, lang=lang, query='Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Bangalore within:400mi ' )

INFO: queries: ['Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Bangalore within:400mi  since:2020-03-01 until:2020-03-03', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Bangalore within:400mi  since:2020-03-03 until:2020-03-06', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Bangalore within:400mi  since:2020-03-06 until:2020-03-08', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Bangalore within:400mi  since:2020-03-08 until:2020-03-11', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Bangalore within:400mi  since:2020-03-11 until:2020-03-13', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:Bangalore wi

In [56]:
banglore_df = pd.DataFrame(t.__dict__ for t in banglore_tweets)

In [58]:
banglore_df.describe()

timestamp_epochs        likes     retweets     replies
count      7.630000e+02   763.000000   763.000000  763.000000
mean       1.586012e+09   347.507208    66.816514   12.385321
std        1.271624e+06   652.932417   103.461458   27.427612
min        1.583037e+09    11.000000     9.000000    0.000000
25%        1.584979e+09    62.000000    19.000000    2.000000
50%        1.586234e+09   126.000000    33.000000    5.000000
75%        1.587142e+09   301.000000    75.000000   11.500000
max        1.587427e+09  5381.000000  1163.000000  365.000000

In [59]:
chennai_tweets = query_tweets(begindate=begin_date, enddate=end_date,limit=limit, lang=lang, query='Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:chennai within:500mi ' )

INFO: queries: ['Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:chennai within:500mi  since:2020-03-01 until:2020-03-03', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:chennai within:500mi  since:2020-03-03 until:2020-03-06', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:chennai within:500mi  since:2020-03-06 until:2020-03-08', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:chennai within:500mi  since:2020-03-08 until:2020-03-11', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:chennai within:500mi  since:2020-03-11 until:2020-03-13', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:chennai within:500mi  

In [61]:
chennai_df = pd.DataFrame(t.__dict__ for t in chennai_tweets)

In [62]:
chennai_df.describe()

timestamp_epochs          likes      retweets      replies
count      2.997000e+03    2997.000000   2997.000000  2997.000000
mean       1.585352e+09     420.062062    166.938939    23.321989
std        1.242494e+06    3039.461395   1671.589350   133.549242
min        1.583027e+09       6.000000      7.000000     0.000000
25%        1.584374e+09      51.000000     22.000000     2.000000
50%        1.585410e+09     100.000000     35.000000     5.000000
75%        1.586355e+09     249.000000     76.000000    14.000000
max        1.587427e+09  151021.000000  60349.000000  5389.000000

In [63]:
kerala_tweets = query_tweets(begindate=begin_date, enddate=end_date,limit=limit, lang=lang, query='Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:kerala within:500mi ' )

INFO: queries: ['Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:kerala within:500mi  since:2020-03-01 until:2020-03-03', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:kerala within:500mi  since:2020-03-03 until:2020-03-06', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:kerala within:500mi  since:2020-03-06 until:2020-03-08', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:kerala within:500mi  since:2020-03-08 until:2020-03-11', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:kerala within:500mi  since:2020-03-11 until:2020-03-13', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:kerala within:500mi  since:

In [64]:
gujrat_tweets = query_tweets(begindate=begin_date, enddate=end_date,limit=limit, lang=lang, query='Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:gujrat within:500mi ' )

INFO: queries: ['Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:gujrat within:500mi  since:2020-03-01 until:2020-03-03', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:gujrat within:500mi  since:2020-03-03 until:2020-03-06', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:gujrat within:500mi  since:2020-03-06 until:2020-03-08', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:gujrat within:500mi  since:2020-03-08 until:2020-03-11', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:gujrat within:500mi  since:2020-03-11 until:2020-03-13', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona  OR coronago min_retweets:10 min_faves:10 near:gujrat within:500mi  since:

In [65]:
kerala_df = pd.DataFrame(t.__dict__ for t in kerala_tweets)
gujrat_df = pd.DataFrame(t.__dict__ for t in gujrat_tweets)

In [66]:
#concatenating tweets of different cities
df = pd.concat([hyd_df, mum_df, banglore_df, del_df, kerala_df, gujrat_df])

In [67]:
df.describe()

timestamp_epochs          likes      retweets       replies
count      1.187200e+04   11872.000000  11872.000000  11872.000000
mean       1.585707e+09     475.225152    137.764319     26.006149
std        1.257154e+06    3453.769391   1180.292508    334.858053
min        1.583036e+09       8.000000      0.000000      0.000000
25%        1.584712e+09      50.000000     20.000000      2.000000
50%        1.585733e+09     101.000000     34.000000      5.000000
75%        1.587028e+09     253.000000     72.000000     14.000000
max        1.587427e+09  151021.000000  60354.000000  19546.000000

Tweets from different cities are collected and concatenated

In [68]:
#
df.head()

screen_name                                     username     user_id  \
0  Clinical_Sim  الطب والمحاكاة #كلنا_مسؤول ... Clinical Sim  2421885032   
1     AlexInAir                                Alex Macheras   191881312   
2    KVishReddy                      Konda Vishweshwar Reddy  1125085620   
3    one_by_two                                  Atul Khatri    28515858   
4    one_by_two                                  Atul Khatri    28515858   

              tweet_id                                 tweet_url  \
0  1234560981763854336  /Clinical_Sim/status/1234560981763854336   
1  1234533775809351681     /AlexInAir/status/1234533775809351681   
2  1234531413191299073    /KVishReddy/status/1234531413191299073   
3  1234515105620144129    /one_by_two/status/1234515105620144129   
4  1234514105102471175    /one_by_two/status/1234514105102471175   

            timestamp  timestamp_epochs  \
0 2020-03-02 19:27:29        1583177249   
1 2020-03-02 17:39:23        1583170763   
2 2020-03-02 17:30:00        1583170200   
3 2020-03-02 16:25:12        1583166312   
4 2020-03-02 16:21:13        1583166073   

                                                text  \
0  ما هي أفضل طريقة لتأهيل الممارسين الصحيين للتع...   
1  Latest: Tajikistan has informed airlines that ...   
2  Thank you Telangana Police for treating me lik...   
3  #Sooryavanshi could only be made with Mumbai P...   
4  #Sooryavanshi trailer with Akshay, Ajay and Ra...   

                                           text_html links  ... has_media  \
0  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...     False   
1  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...     False   
2  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...      True   
3  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...     False   
4  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...     False   

                                            img_urls video_url likes  \
0                                                 []              20   
1                                                 []              38   
2  [https://pbs.twimg.com/media/ESHvxbDU8AEMq3U.jpg]             631   
3                                                 []            1674   
4                                                 []             583   

   retweets  replies  is_replied  is_reply_to  parent_tweet_id reply_to_users  
0        52       30        True        False                              []  
1        24        1        True        False                              []  
2       176      111        True        False                              []  
3       185       16        True        False                              []  
4        56       14        True        False                              []  

[5 rows x 21 columns]

In [70]:
df_dup=df.drop_duplicates(subset ="tweet_id", keep=False,inplace=True) 

In [72]:
df.describe()

timestamp_epochs         likes      retweets      replies
count      2.375000e+03   2375.000000   2375.000000  2375.000000
mean       1.586487e+09    386.436632    152.181053    17.922526
std        1.115549e+06   1586.129907   1393.005278    65.215190
min        1.583043e+09     10.000000      0.000000     0.000000
25%        1.585553e+09     50.000000     18.000000     1.000000
50%        1.587115e+09     99.000000     30.000000     4.000000
75%        1.587329e+09    232.000000     60.500000    11.000000
max        1.587422e+09  31800.000000  60354.000000  1197.000000

In [73]:
df.head()

screen_name               username              user_id  \
32      mbah_mijan             Mbah Mijan            227961909   
53   SriLankaTweet     Sri Lanka Tweet 🇱🇰             41786801   
94     babblingnia  Nia Bebel #StayAtHome  1131926153474654209   
124  neyrashazeyra      Penjual Sanitizer             22664411   
130       yaashjay                  yaash           1210815386   

                tweet_id                                  tweet_url  \
32   1234382734123917312     /mbah_mijan/status/1234382734123917312   
53   1233998380398972928  /SriLankaTweet/status/1233998380398972928   
94   1236217732065939456    /babblingnia/status/1236217732065939456   
124  1235823254268329985  /neyrashazeyra/status/1235823254268329985   
130  1235747618187141120       /yaashjay/status/1235747618187141120   

              timestamp  timestamp_epochs  \
32  2020-03-02 07:39:12        1583134752   
53  2020-03-01 06:11:55        1583043115   
94  2020-03-07 09:10:49        1583572249   
124 2020-03-06 07:03:18        1583478198   
130 2020-03-06 02:02:45        1583460165   

                                                  text  \
32   Virus Corona positif masuk Indonesia, dan vaks...   
53   What we know #LKA  #CoronaVirus.\n2 Lankans ar...   
94   Been using these for yearssssss bila everytime...   
124  Mydin jual mask ke kertas ni? pic.twitter.com/...   
130  Corona Virus ? Try our smoke ball. If you use ...   

                                             text_html  \
32   <p class="TweetTextSize js-tweet-text tweet-te...   
53   <p class="TweetTextSize js-tweet-text tweet-te...   
94   <p class="TweetTextSize js-tweet-text tweet-te...   
124  <p class="TweetTextSize js-tweet-text tweet-te...   
130  <p class="TweetTextSize js-tweet-text tweet-te...   

                                                 links  ... has_media  \
32                                                  []  ...     False   
53   [https://www.lankaeducation.com/wuhan-coronavi...  ...      True   
94   [https://twitter.com/hannabunnyb/status/123617...  ...     False   
124                                                 []  ...      True   
130                                                 []  ...     False   

                                              img_urls video_url likes  \
32                                                  []             140   
53   [https://pbs.twimg.com/media/ESAK-qBXYAAAPv6.jpg]              73   
94                                                  []             797   
124  [https://pbs.twimg.com/media/ESaGr6KUUAAn9Kq.jpg]             133   
130                                                 []             293   

     retweets  replies  is_replied  is_reply_to  parent_tweet_id  \
32         43       18        True        False                    
53         40        3        True        False                    
94        309        5        True        False                    
124        94       14        True        False                    
130       214        6        True        False                    

    reply_to_users  
32              []  
53              []  
94              []  
124             []  
130             []  

[5 rows x 21 columns]

In [74]:
df2 = pd.concat([hyd_df, mum_df, banglore_df, del_df, kerala_df, gujrat_df])

In [78]:
df2=df2.drop_duplicates(subset ='tweet_id')

In [79]:
df2.describe()

timestamp_epochs          likes      retweets       replies
count      5.742000e+03    5742.000000   5742.000000   5742.000000
mean       1.585956e+09     444.274643    144.626611     23.352490
std        1.251563e+06    2907.280274   1306.269770    278.483762
min        1.583036e+09       8.000000      0.000000      0.000000
25%        1.584894e+09      49.000000     19.000000      2.000000
50%        1.586091e+09     100.000000     33.000000      5.000000
75%        1.587142e+09     243.750000     68.000000     13.000000
max        1.587427e+09  151021.000000  60354.000000  19546.000000

In [80]:
df2.head()

screen_name                                     username     user_id  \
0  Clinical_Sim  الطب والمحاكاة #كلنا_مسؤول ... Clinical Sim  2421885032   
1     AlexInAir                                Alex Macheras   191881312   
2    KVishReddy                      Konda Vishweshwar Reddy  1125085620   
3    one_by_two                                  Atul Khatri    28515858   
4    one_by_two                                  Atul Khatri    28515858   

              tweet_id                                 tweet_url  \
0  1234560981763854336  /Clinical_Sim/status/1234560981763854336   
1  1234533775809351681     /AlexInAir/status/1234533775809351681   
2  1234531413191299073    /KVishReddy/status/1234531413191299073   
3  1234515105620144129    /one_by_two/status/1234515105620144129   
4  1234514105102471175    /one_by_two/status/1234514105102471175   

            timestamp  timestamp_epochs  \
0 2020-03-02 19:27:29        1583177249   
1 2020-03-02 17:39:23        1583170763   
2 2020-03-02 17:30:00        1583170200   
3 2020-03-02 16:25:12        1583166312   
4 2020-03-02 16:21:13        1583166073   

                                                text  \
0  ما هي أفضل طريقة لتأهيل الممارسين الصحيين للتع...   
1  Latest: Tajikistan has informed airlines that ...   
2  Thank you Telangana Police for treating me lik...   
3  #Sooryavanshi could only be made with Mumbai P...   
4  #Sooryavanshi trailer with Akshay, Ajay and Ra...   

                                           text_html links  ... has_media  \
0  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...     False   
1  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...     False   
2  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...      True   
3  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...     False   
4  <p class="TweetTextSize js-tweet-text tweet-te...    []  ...     False   

                                            img_urls video_url likes  \
0                                                 []              20   
1                                                 []              38   
2  [https://pbs.twimg.com/media/ESHvxbDU8AEMq3U.jpg]             631   
3                                                 []            1674   
4                                                 []             583   

   retweets  replies  is_replied  is_reply_to  parent_tweet_id reply_to_users  
0        52       30        True        False                              []  
1        24        1        True        False                              []  
2       176      111        True        False                              []  
3       185       16        True        False                              []  
4        56       14        True        False                              []  

[5 rows x 21 columns]

In [81]:
#Collecting more tweets
telangana_twts = query_tweets(begindate=begin_date, enddate=end_date,limit=limit, lang=lang, query='Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona OR hyderabad OR people OR coronago min_retweets:10 min_faves:10 near:Telangana within:500mi ' )

INFO: queries: ['Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona OR hyderabad OR people OR coronago min_retweets:10 min_faves:10 near:Telangana within:500mi  since:2020-03-01 until:2020-03-03', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona OR hyderabad OR people OR coronago min_retweets:10 min_faves:10 near:Telangana within:500mi  since:2020-03-03 until:2020-03-06', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona OR hyderabad OR people OR coronago min_retweets:10 min_faves:10 near:Telangana within:500mi  since:2020-03-06 until:2020-03-08', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona OR hyderabad OR people OR coronago min_retweets:10 min_faves:10 near:Telangana within:500mi  since:2020-03-08 until:2020-03-11', 'Corona OR Coronavirus OR Police OR mask OR #corona OR #gocorona OR hyderabad OR people OR coronago min_retweets:10 min_faves:10 near:Telangana within:500mi  since:2020-03-11 until:2020-03-13', 'Corona OR Cor

In [83]:
tel_df = pd.DataFrame(t.__dict__ for t in telangana_twts)

In [84]:
blr_twts = query_tweets(begindate=begin_date, enddate=end_date,limit=4000, lang=lang, query='min_retweets:10 min_faves:10 near:Bangalore within:500mi ' )

INFO: queries: ['min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-01 until:2020-03-03', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-03 until:2020-03-06', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-06 until:2020-03-08', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-08 until:2020-03-11', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-11 until:2020-03-13', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-13 until:2020-03-16', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-16 until:2020-03-18', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-18 until:2020-03-21', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-21 until:2020-03-23', 'min_retweets:10 min_faves:10 near:Bangalore within:500mi  since:2020-03-23 until:2020-03-26', 'min_retweets:10 min_faves:10 near

In [85]:
blr_df = pd.DataFrame(t.__dict__ for t in blr_twts)

In [86]:
data = pd.concat([df2, tel_df, blr_df])

In [87]:
#As the scraper also generates duplicate values, we are removing the duplicate values
data=data.drop_duplicates(subset ='tweet_id')

In [88]:
data.describe()

timestamp_epochs          likes      retweets       replies
count      1.048300e+04   10483.000000  10483.000000  10483.000000
mean       1.585654e+09     386.286845    147.298674     23.086616
std        1.314549e+06    2344.392119   1252.141364    217.155543
min        1.583031e+09       3.000000      0.000000      0.000000
25%        1.584591e+09      47.000000     20.000000      2.000000
50%        1.585666e+09      97.000000     35.000000      5.000000
75%        1.587021e+09     237.000000     73.000000     14.000000
max        1.587427e+09  151021.000000  60354.000000  19546.000000

In [89]:
data['retweets'].value_counts

<bound method IndexOpsMixin.value_counts of 0        52
1        24
2       176
3       185
4        56
       ... 
4172     30
4173     35
4174     18
4175    741
4176     20
Name: retweets, Length: 10483, dtype: int64>

In [90]:
data.to_csv (r'C:\Users\vamshi\Documents\Scrape\tweets.csv', index = False, header=True)

In [103]:
user_name_list = data['screen_name'].t

In [104]:
user_name_list = list(user_name_list)

In [105]:
print(user_name_list)

TypeError: cannot do slice indexing on <class 'pandas.core.indexes.range.RangeIndex'> with these indexers [['Clinical_Sim', 'AlexInAir', 'KVishReddy', 'one_by_two', 'JebiMather', 'NNN_TNP', 'prasarbharati', 'YohanesPatty2', 'thehaelyn', 'SAFE_JOHNY', 'srijanpalsingh', 'Zayed_U', 'manateedugong', 'mbah_mijan', 'ShaikhHamzaIMS', 'IamOnir', 'RiaRevealed', 'CPEC_CSCEC', 'dbeasley1', 'svirtuoso2010', 'scribe_prashant', 'PuneAkkians', 'IamNaveenKapoor', 'DelhiHeritage', 'AdnanSamiLive', 'zfrmrza', 'mehdizafar', 'jayant_rokade', 'TonsTweetings', 'taneja_devander', 'mahaabvp', 'bangtanUAE', 'somyalakhani', 'SriLankaTweet', 'deepak_mragwani', 'IAmSudhirMishra', 'DrAMSinghvi', 'sarnab_', 'SarikaPanda', 'RichardBarrow', 'K4kaviking', 'YallaIsraelSUN', 'DuchessT_', 'axanner', 'NashriqFazi', 'BrajeshOfficial', 'usmanshehzad90', 'AmeenaAna', 'jgopikrishnan70', 'RoadsOfMumbai', 'rameshlaus', 'sidhant', 'zeeshan_iyc', 'TaqsNews', 'babblingnia', 'RoshanSdrprop', 'Teensthack', 'MoCA_GoI', 'TheGautamGulati', 'WhereIsMy_Food', 'VapiAshram', 'NarsiOm', 'KrishnaMohanSha', 'MUMBlr1', 'Mohammad_86_86', 'aphradity', 'sudarsansand', 'samyak_samaj', 'sana_kujira', 'Sheeshpal_Yog', 'actor_shirish', 'MehHarshil', 'alihashem_tv', 'SanyogeetaSinha', 'Ach_Balkrishna', 'ItalyinIndia', 'neyrashazeyra', 'NileshShah68', 'arnauldmiguet', 'Amit_Jeswani1', 'yaashjay', 'Rashed_pti', 'SumetaSyed', 'GuftarAhmedCh', 'paneeripatakhay', 'attorneybharti', 'rashidrafiq1975', 'UmarKhalidJNU', 'connectgurmeet', 'prajwalmanipal', 'nellyellyana', 'sukh_slatch', 'SohaTazz', 'pandu_kdp', 'KOLHAPUR_POLICE', 'maldhaif1', 'AudiPhotography', 'roxannebamboat', 'DarshanDevaiahB', 'GaladariMD', 'JalalQazi', 'nate_bowling', 'sonumpun', 'shreemiverma', 'madhukishwar', 'NBirenSingh', 'Shalinisingh315', 'AIiizzath', 'nsptsaiphanitej', 'nagarjund', 'Sajad_Karwan', 'MrinalPande1', 'EedhamRasheed', 'somecityusa', 'OwToTheZie', 'prasanto', 'dibbi18', 'naecifer', 'drashwinitaur', 'K_DeLanerolle', 'Mansoorufone', 'muralwrites', 'ArchisMohan', 'BMTheEquityDesk', 'AliHashimSmith1', 'mobeid', 'niyaf', 'saud_alattiyah', 'UdaySRana', 'NayabGJan', 'sayantansunnyg', 'therealdaddymo1', 'Khongsak', 'StonerJesus', 'BLRrocKS', 'Ash_Stewart_', '151985Lucky', 'umashankarsingh', 'pratyushdixit', 'DrPascalMeier', 'jahmed95', 'Sangrisaeed', 'mirrorCracking_', 'munza14', 'qnqamar', 'kartikeya_1975', 'INasser_Mohmmed', 'iFaridoon', 'SinghLions', 'NiyazYoosuf', 'IIMAhmedabad', 'karunasagarllb', 'KartikPatadia69', 'NishaPurushoth2', 'noorsheerr', 'ddanaalm', 'theabbasvlog', 'anantjain1989', 'Sirjana_Singh', 'carlwheezersbby', 'KocharSundeep', 'Javidfaisal', 'shaileshkpandey', 'saahilmenghani', 'vikrantgupta73', 'anilvijminister', 'gandabherunda', 'suyu_pmr', 'RajitaBagga', 'Mubeen_Ul_Haq', 'JorArguello', 'UsherKomugisha', 'rubenmasc', 'RaunakRK', 'p_ghildial', 'LangaMahesh', 'OfficeofUT', 'AmberRShamsi', 'Sujan_Speak', 'iamkash_kr', 'asifkhanmzn', 'deepaliranaa', 'George_Orozco', 'SoodRajive', 'StoMedical', 'taimoormaheraly', 'ImranAlikeerio', 'pelolesskinco_', 'DalrympleWill', 'joshijikabetaa', 'heynixhant', 'gulshandass2708', 'CikManisDonut', 'gad_media', 'rajtoday', 'manukumarjain', 'BodkheShilpa', 'Santosh34645481', 'BucketheadCase', 'pushkarbendre', 'Manaramdas56', 'CharanT16', 'HOPEFULSEOKIE', 'KunalV3rma', 'iamramizxrami', 'AmritaSTiwari', 'RahmayadiEdy', 'softness_alert', 'ProfYLH', 'grvpugilist', 'NeerajK35713705', 'DigvijaynathTi4', 'ari_maj', 'dwallbank', 'KarnikaKohli', 'Orlygoldschmidt', 'AshokShrivasta6', 'jethanandani14', '__nynaRSLN', 'itssinghswati', 'CynthiaDRitchie', 'spreadjainism', 'astrosumitbajaj', 'CommissionerSMC', 'nandita_venky', 'rohansmitra', 'warunchit106', 'alok_ajay', 'CharuPragya', 'arpitjain__', 'Prasad_Darling', 'HARISHMALIK007', 'KirenRijiju', 'Cuckoo1985', 'EevaPaavilainen', 'maryamhome', 'DwivediSurbhi', 'DarshakHathi', 'AatifWaqar', 'radhacharandas', 'muannas_alaidid', 'sureshpprabhu', 'vijayrpandey', 'yusufmurxd', 'AliKhizar', 'aishe_ghosh', 'f4izalhassan', 'NasirKhuehami', 'vinod_bansal', 'raghvanvenkat', 'RawitHanutsaha', 'RaniRan60710022', 'AnimaSonkar', 'K26s9v56', 'RSupercop', 'iyadavsena', 'pallavict', 'DelphineEvelyn1', 'sahneydeepak', 'mayankjain100', 'VishweshwarBhat', 'rexrao52', 'essel1', 'Shamsher_IPS', 'Avadhutwaghbjp', 'mumbaimatterz', 'SecyChief', 'BeingNafeel', 'sunilvikas', 'pooja_news', 'vishnupancholi3', 'mYpO8kRaerykimA', 'Smitdesai8320', 'DeepalTrevedie', 'sshikhar33', 'AkshaySiliguri', 'NashikNews', 'rohanduaTOI', 'DGP_Goa', 'LeenaVaishno', 'SreenitaGoenka', 'AnilKapoor', 'anandrai177', 'VarinderKhosa77', 'iabhishekpandya', 'MdFujailAhmed', 'anilsharma45', 'IndurChhugani', 'drqayumiitk', 'MimiSayani', 'kAfghaan', 'mehran_narejo', 'sheryarbhagat', 'Runjhunsharmas', 'amitranabjp', 'chandpc123', 'devduttmyth', 'PKRamanathanBPC', 'VidadalaRajini', 'musharraf_ias', 'rkhuria', 'GulamRabbaniAdv', 'AkramShahBJP', 'sunandasaxena', 'Arshad_Afridi', 'amarprasadreddy', 'TheSaudiHeart', 'preetikhatri22', 'kbkindia', 'SureshAngadi_', 'thakur_shivangi', 'BBCsarika', 'sivakumarramak4', 'Maaachaaa69', 'HarbirSinghSuri', 'AleemIlahi', 'sitaramlamba', 'SmartPune', 'juhiesingh', 'DrKirodilalBJP', 'gauravcsawant', 'imtiyazuk', 'gopalkagarwal', 'SpeakingShiva', 'brendagoh_', 'NazzyFuzzy', 'MSApunya', 'AmarUjalaNews', 'moeager', 'skynewsarabia', 'AJEnglish', 'febrofirdaus', 'SBengali', 'dhpierson', 'bernamadotcom', 'STcom', 'Mr_703n4_______', 'jeffsekerak', 'TV9Bharatvarsh', 'DDNational', 'wehlie_ahmed', 'UsamaQureshy', 'shoaibsb1', 'Imaldris1', '8882259371', 'ecstacy_ded', 'radhika1705', 'AbdulBillowAli', 'MaryMmboyi', '7attu', 'buriroGM', 'silarindam', 'DevineniUma', 'GandapurPAT', 'DrGarekar', 'Niveshrau', 'putrajayacorp', 'dilipjoshie', 'PawanDixitINC', 'ganeshmpandey', 'ArsalanGhumman', 'KasimGillani', 'kashifshoroppp', 'NouraAlKaabi', 'Madan_Chikna', 'imAditiofficial', 'shammybaweja', 'sbksinghips', 'vbwalia', 'ganant', 'sarthakkashyap4', 'The_NehaJoshi', 'rohit_chahal', 'AmitHPanchal', 'amitbhawani', 'PoliceYnr', 'Rajendr65303329', 'bhimbali_bhanu', 'DasarwadD', 'pksahukar', 'Satishrathod100', 'Sufiyan_INC001', 'Inspshishpal', 'navaidhamid', 'DHA_Dubai', 'salimkaskar3', 'vikasvaibhavips', 'cardio73', 'suni_krishna', 'ParinSays', 'KishoriPednekar', 'REBELKAUR', 'PardeepParihar', 'HarishSGariya', 'AkritiBhatiaa', 'rajudasonline', 'mahajan1bhushan', 'saprajapati1', 'vineethmore', 'maulinshah9', 'anandaditya', 'ramanmann1974', 'Indian_pandit_', 'ThePolKholShow', 'tamilgramsi', 'manav700', 'mpopat', 'Ayodhyawasi_', 'praveenskapoor', 'catsabacus', 'manishrath', 'i_am_srk_jawwad', 'PrasadLadInd', 'ishafoundation', 'MayankYadavAdv', 'ImWaqar14', 'RONBupdates', 'AAPChhattisgarh', 'Bazophil', 'deeptivishalma1', 'worldNTRfans', 'surendrasnaagar', 'amethipolice', 'priyagupta999', 'barkhatrehan16', 'indyrasheed', 'Yog_Singh_', 'raghunathan_ps', 'digvijayvasuni1', 'DodaniRaji', 'bk69321288', 'RekhaBansalMom', 'yimsterz', 'ibrohi31', 'MohammadMohaaa', 'revolution_raj7', 'SaleemFarrukh', 'Rajan_Medhekar', 'Wassup_Mumbai', 'Myself_Jaanu', 'saeedalhamed_', 'MEDENT_Official', 'Hoimee', 'SanjoyRoyTWA', 'pranavanandaji', 'kanishkajadhav', 'ZubairJamaliPK', 'tennis_meenal', 'AttshanAliAbbas', 'rais_shk', 'Boxerpanghal', 'saumyasharma711', 'DineshKPatnaik', 'Reema_ahluwalia', 'Miesedapkuahhh', 'flyingbeast320', 'thekiranbedi', 'archanatambade', 'gabrielbeblo', 'Shivnand_Prabhu', 'ajayacharya', 'GajenderInsan', 'ProfMIsmail', 'AamirSuhailAAP', 'actoranirudha', 'stagecoremv', 'eddydaud', 'ZameerMaan', 'SindhCMHouse', 'amnan1drus', 'faziljamili', 'AhmadWaleed', 'iankitvora', 'anil_hebbar', 'jdpatel900', 'head_of_toys', 'Animesh67811557', 'MelGohCNA', 'Hasni_Johor', 'Ayab_Ahmed', 'Humour_lesss', 'KeshooBai', 'PNRai1', 'QauVines', 'hussain_imtiyaz', 'minitinnaik', 'Seoul_Cafe', 'aaliffasri_', 'yahhhhhkhai', 'monidipadey', 'roopenshu', 'GhanwerIsran', 'aliaftabsaeed', 'takterqyira', 'RitiPathakSidhi', 'sariyarana', 'AvdheshanandG', 'myjoe_', 'MNSPrashantRane', 'sanghaviharsh', 'TeamQuetta', 'cricketaakash', 'mihirkotecha', 'bhuvikal', 'XuZeyu_Philip', 'RpRamesh10', 'GhazaleeIbr', 'xzAmyxz', 'bakure17', 'ouinokdum', 'GujHFWDept', 'RitaBJoshi', 'sajithpremadasa', 'Vyas241S', 'jogender_emc2', 'ghauraghaura', 'asifnz', 'therban14amit', 'MawraHocane', 'sribasavaraju', 'drSujayShad', 'arhuml92', 'keswins', 'DrVikasPadha', '_deheyyyy', 'achlendra', '__akmalazman', 'CSKFansOfficial', 'mujahidikhlaq', 'mandellivaleria', 'JanbazK786', 'AmSyedaFatima', 'Shiddane', 'MoHADM_Somalia', 'ImranHussaain', 'faliqfahmie', 'agscinemas', 'zimizzzz', 'CoachHesson', 'faizyology', '_ashokpareek', 'ikhwanzrf', 'Aqeelhashmi37', 'AzamJoyoPPP', 'NKMalazai', 'roseayda', 'macharajarao', 'dnlimn_', 'forfunbangkok', 'SaleemulHuq', 'PirzadaPP212', 'JSPLFoundation', 'Sachin_Agr', 'PejuangJlnLurus', 'AsadAToor', 'debabrata2008', 'Harshach3883', 'delamadridrob', 'muharikah', 'irfandinzly', 'ainunnajib', 'dan_murphy', 'Qadri555', 'iamtkumaar', 'adwaitdeshpand', 'wiratmadinata98', 'rajini_balu', 'Dikshitsdoshi', 'iamanilshetty', 'aaigoaairport', 'UNICEFIndia', 'AoffyMyname', 'o_positive_', 'purveshsarnaik', 'Nishant_Mishra9', 'HariOm_o3', 'SinghAs681', 'Munilokesh', 'Gurkirtan_77', 'SyedAminulHaque', 'SumitraAAP', 'khan_haq', 'khurshid071', 'SaifulJamia', 'Jishnu_Devvarma', 'alizia05', 'janaram24', 'spoudel2076', 'spirittaruvar', 'ruffleseed', 'sadiq_zaf', 'rasyidnorrizan', 'idnani_nandini', 'IamAnilBhasin', 'Roarlk', 'drkiritpsolanki', 'smitadeshmukh', 'SikanderBalouch', 'LotusTowerCMB', 'dnetta', 'nel_zl', 'AshishSinghLIVE', 'mkvenu1', 'AdamRBond', 'ChitnisArchana', 'ArifRafhanDraw', 'gormilhazhrana', 'kelvinobioma3', 'REALsaeedajmal', 'muruga_TNIE', 'brizpatil', 'aawasthilaw', 'SonnalliSeygall', 'DrTamilisaiGuv', 'arunkumarkundra', 'sinlbsa', 'AipcFbd', 'burhaankinu', 'Dineshbaghelbjp', 'nagarkoti', 'pauriahcarey', 'RudraVS', 'VishveshVVv', 'mymalishka', 'DrSaurav5', 'adityasahu955', 'ChouhanShivraj', 'nusrat_anis', 'achyuta_samanta', 'i_simab', 'Milan_reports', 'Abhijitdongre99', 'akbar62', 'khan_here_', 'asifsuhaf', 'KhadimH21887072', 'IMFALtd', 'PuspaManiKharal', 'Kumkum26', 'Being_Akbar', 'rajansi45', 'hemasthakur', 'sandeep04CBS', 'idiamondbabu', 'Amitkalraj', 'Shekharyadav02', 'kayjay34350', 'Me_Saleel', 'sachinbambhniya', 'HitendraVora2', 'SnehalDoke', 'ImArjun_M2R', 'meanshuman', 'ShagunS28432208', 'mybmcWardGS', 'sanju_gouda', 'Hendry7715', 'rajeshmadaan13', 'NareshSaur', 'Asyeedhamdan', 'shahbelladonna', 'VictorKamhuka', 'Jayaasnani2', 'Pk_kaurav', 'NadzimAizat', 'CtBaroda', 'thaimoph', 'Kishore36451190', 'NaveenMIRROR', 'markandeysir', 'shenoy70', 'Guttajwala', 'KosalramT', 'ShivrajKaBhanja', 'LoJcp', 'VandanaSsingh', 'goelgauravbjp', 'rakeshkhulbe', 'pokrath', 'MukulAgarwal66', 'RKarmaliani', 'deepaksomisetty', 'Halawala', 'nsvirk', 'ArogyaAndhra', 'HUNARGUPTABJP1', 'MrRathna', 'Geetu_Moza', 'MJBank', 'Noorulamin000', 'tamkannan', 'abhiemanyu', 'Viv2511', 'rahullko173', 'vivaceAjay', 'amitabhprevi', 'azam_twitz', 'KALLIVAYALIL', 'ahiteja', 'nandan_rss_up', 'AmirMateen2', 'faisalahmadj', 'aamir_ibrahim01', 'NidhiKNDTV', 'IntanKaharuddin', 'ibzzaz', 'ariffazam_', 'bbsranjan', 'DrJasonLeong', 'jkmultiplus', 'vegevenkatvijay', 'AskRaushan', 'Chelsea_Ngente', 'YasserRahman', 'omaryonce', 'KunalChoudhary_', 'corona_virush', 'anil_prongs', 'mkoirala', 'khiltasooraj', 'Bhushansz', 'AfidaSohana', 'syedharisahmed4', 'TheShuchiSingh', 'GopalSinghINC', 'razieques', 'SecySanjay', 'infoPKU', 'Arie_BangsaOne', 'MyLovePeony', 'HSajwanization', 'RajaFahadAliKh2', 'manojmuntashir', 'VivanMarwaha', 'pujanbiswaspyc', 'StefynaEmily', 'ImSuresh_insan', 'amit0917', 'iCandy_pw', 'navvyB', 'shivangimrs', 'TIIIIIIIIMOTHY', 'LatoyaFerns', 'kirubamunusamy', 'kdevforum', 'Lkamikaze_', 'JhaSanjay', 'KamranKhanZeb', 'rupasubramanya', 'Dr_abdoty', 'virendra79', 'AashifQureshi9', 'panhwar_noman', 'Rabs_AA', 'Ltgen_Ashok', 'meorzmeoraz1', 'dyradyre', 'ADChanna', 'SAvgoustides', 'AbijitG', 'AhsanullahSaid', 'LaLaMuneer33', 'varshasinghs', 'ITheTaurus', 'karem_ayala', 'anshultiwaribjp', 'PrasadKarwa', 'RaiSahab20', 'TedhiLakeer', 'amnakhani123', 'Breathedreamgo', 'rogerkver', 'AmulThane', 'TcGehlotOffice', 'jackie14AP', 'SediqSediqqi', 'drdubeysushil', 'bhavikajTOI', 'navift', 'fxkhrudeen', 'debbiediscovers', 'sharma_jyo', 'BiswarupG_ET', 'payalmehta100', 'Ibrahimnolosha', 'RCBTweets', 'Arshisarunfan', 'anoojpal', 'Shubham2ag', 'thisisridz', 'antoantonyinc', 'xbobongx', 't_047bkk', 'arjyabratapande', 'AatifAzio1', 'NoiritaRehman', 'gauri_maulekhi', 'shri_siddh', 'RashadMaqsood', 'nnojha', 'diljitdosanjh', 'AshwDeshpande', 'RiyakhanMNS', 'MASSAJITH', 'ulaganayagan1', 'AbdullahDayo', 'Sujata_Padhy', 'Srk_bangalore', 'BagdeShan', 'binaniyash4', 'wellbeingvoyage', 'RitikGupta1999', 'anupamachopra', 'ruhail11', 'aniruddh_dave', 'prempanicker', 'logicalindianz', 'ReporterVikrant', 'rajusrinivas11', 'Rajnees39139102', 'sandep_lnsan', 'Abhadra7', 'Chaiti', '40Mns', 'nadeeminc', 'RoPJRJonesRobin', 'DeeptimanTY', 'jehan_hameed', 'WaqasGujjar229', 'gulraizkhan', 'SevadalUKP', 'iam_farha', 'gaj_2', 'WHOOmanOffice', 'limbani_akash', 'tondbz2', 'AlkhidmatOrg', 'meevkt', 'ZarbeKaleemi', 'dhananjay_munde', 'nazimsat', 'Sandeep29032017', 'mandal9938', 'SandeepUnnithan', 'CA_Deepanshu_G', 'iMonkJi', 'Shaikahmedsaud8', 'IshitaYadav', 'isai_', 'swarupdas', 'nasiransari2010', 'MdPvvnl', 'pankhuripathak', 'Ksf_alvi', 'amitabhnews18', 'TheBanat', 'i_m_dwirohit', 'TheSikhBlood', 'AMBAbsolute', 'BhagwaBalak', 'TN_PYC', 'rameshnarayan', 'RanjeetOraon100', 'udsankar', 'jbhavan', 'antaintiom', 'chaminda_vaas', 'IamUbaidSheikh', 'OBP_News', 'plpunia', 'Direct_Dil_se_', 'SultanAlQassemi', 'gunjakapoor', 'saurabh_1993a', 'umairahmadjui', 'sherajai', 'harshcha', 'gamikgupta', 'PrashanthSarma', 'IhsanMarwat_786', 'SP_Amreli', 'AnkitSi94964872', 'VrushantM', 'ShainaNC', 'sami_journalist', 'Akshita_N', 'akashtomarips', 'LahriSunil', 'funchershop', 'wahidhalepoto', 'nolanentreeo', 'jagdishmukhi', 'geddiejdk', 'siddaramaiah', 'irajashekaran', 'sudhirpandey851', 'concept_15', 'modimanju1', 'jainendrajeevan', 'cbhanp', 'Ccpur_Police', 'gajendrasahu21', 'Bhupesh1995pah1', 'ajay53om', 'anupamnawada', 'PrtGuberanPMK', 'PoliticalShiaPK', 'ArijitmINC', 'PramodChturvedi', 'tamizh_i', 'docvjg', 'pp202pmln', 'im_ankitb', 'shraddhadas43', 'Aastha9305', 'YogeshKale19751', 'sarahyasminazmn', 'Huzaifa_Aamir', 'faghahdieanna', 'KFoodhaadhi', 'SP_Atif_Nazir', 'Reshu1977', 'Real__Akash', 'journovivek', 'KnottyCommander', 'IacGaurav', 'AdityaTrivedi_', 'UditMathur', 'mukmin_farisi', 'dineshjain0625', 'imparixit', 'archanatriv', 'BosJohor2010', 'SaifBhatti_MSM', 'Chumairkamboh4', 'Devvrat33431231', 'calamur', 'dr_m_kamran', 'HarmanManchanda', 'SEjaharali', 'zalmayzia', 'ManishThekda', 'harishvnair1', 'pafcoms', 'Micha_Alvez', 'Sidbakaria', 'shri8354', 'ddmallick', 'StocksResearch', 'Paak_Mohobbat', 'ICCR_Delhi', 'UnbeatableFaibi', 'PrabhuChawla', 'AshishSainram', 'LubnaZahir', 'amanashraff', 'virendradubey86', 'myplacenhistory', 'HarshVardhanTri', 'mahin2000', 'Sandeep_1966', 'toxicanrajput', 'Francis_Joseph', 'nkaggere', 'SEKHAR_JSP', 'iOldMonk', 'FMumbaiker', 'p_jaikumar', 'i_cp_yadav', 'Gourayy', 'poornima_mishra', 'MalaysianRelief', 'SeharKamran', 'americanbadu', 'madehanaqvi', 'Lostsoul_iQu', 'Shanirss', 'S21Nawab_', 'Santhos05987868', 'iloveqatar', 'ShailyTiwari5', 'rishikant1994', 'GabbarSanghi', 'dheeraj_fartode', 'ibrijmohanJi', 'stoppression', 'jnbeuze', 'brijeshkalappa', 'theRohitBansal', 'kinjalmishra211', 'arunp2810', 'ReddyNsui', 'JivanPachpute', 'TalibHashem', 'ImRaina', 'HarbingerAFC', 'ZainHQ', 'SJzehri', 'zaidia', 'Himanshu_Aajtak', 'singh_biswajeet', 'hdhingra', 'manojjoshiddn1', 'VasanMSV', 'Amitava_BJP', 'Shashanka_IAS', 'iskandarsamad', 'SamSRKian_31', 'bajrangi4bjp', 'lachvi', 'tirupati_balaj', 'ethicalsid', 'Mayorspeaks', 'NaveedRazzaqPTI', 'WHOMaldives', 'TerasinR', 'AaliHasan10', 'Zain_Journalist', 'iYashwantJadhav', 'raveena116', 'asyrfzainal', 'MutahNapoleon', 'JunaidkhanREAL', 'KodgireKedar', 'Nitikasharmabjp', 'dusktillfajar', 'HydUniv', 'pinku74patel', 'swadesh171', 'beingpriyanka15', 'harismurali', 'akimmich_', 'foodietweeter', 'MsKajalAggarwal', 'Andrew007Uk', 'SudhinBhadoria', 'ParasharGanesh', 'MrPrestante', 'Simwal', 'jogalshailaja', 'AvdheshAvasthi1', 'HassanJehangeer', 'iamhoneygrover', 'MaheshM98661024', 'vaishna68850082', 'DonapatiVishnu', 'vinuthewriter', 'tedregencia', 'porinju', 'imranazlan_91', 'AnuradhaSKurmi', 'g_l_prasad', 'SapruAndRao', 'AkantoSamiul', 'vihangsarnaik', 'javed_deziner', 'rafidah72', 'HimeshMankad', 'MirchiShwetima', 'bobby_yedida', 'Mustafaa7', 'SwamiBrahmachit', 'jalajboy', 'nairsameer', 'MSG__MYJAAN', 'BangashYK', 'sapnamadan', 'activistritu', 'mahesh3419', 'iindrojit', 'adaazam', 'SBM_4007', 'agrapolice', 'AnilDass211', 'karanbjphp', 'IrfanPathan', 'Divyeshsahu5', 'suvirsaran', 'SUNITAVYASBJP', 'pspoffice', 'TariqJamilOFCL', 'Actor_Mahendran', 'ganeshjoshibjp', 'khattaAngoor', 'SattarFarooqui', 'kashifmughal', 'DipikaPS', 'mhidayatullah31', 'jitkasame', 'DrJitendraSingh', 'BTVI', 'rajbirrohilla', 'kailashandani', 'Deepmalatandon2', 'TRBRajaa', 'kalpna02891469', 'gdalmiathinks', 'firozyehya', 'GopalKeshri', 'MyHEALTHKKM', 'Tarikkhattak', 'annusraza', 'Sowmyareddyr', 'vbzu', 'OGboy__', 'jitendradixit', 'karantacker', 'M4lev0lent', 'RevathyAshok', 'mintu2318', 'Drsubhassarkar', 'faizalhamssin', 'rahatindori', 'Krutagyta', 'Nagercoil_SFC', 'AdilKhanAAP', 'arvindpadhee', 'PavanRanaRSS', 'LumosMaxima1510', 'mohamed_isa', 'PankajJainClick', 'iAjayThakur', 'bnodg', 'chetanaupadhya2', 'WordsSlay', 'amitabhk87', 'fit_yh', 'sabbybeautyhub', 'DhumalSpeaks', 'MechDara1', 'dhritrastra', 'trichyls', 'Abuhudzaifah82', 'DrFsalman', 'AishaFSarwari', 'NoorNugali', 'sn_sabat', 'RajkhowaRahul', 'AZK_KHAN_VIP', 'samidhas', 'Mushahid', 'LaghateParthOfc', 'amaazmi', 'SikanderFayyaz', 'atulagrawwal', 'AssadAbro', 'spgroff', 'kawinpt', 'fawzie_UL', 'Doctor_M8', 'Buntyshelke_inc', 'BrazilEmbassyIN', 'iammunishankar', 'rahulsmane', 'abhijit227', 'PiyushsainiBjp', 'liparevijay', 'kilari_murli', 'scpatodia', '_RG_P', 'EngChemi', 'iamAcax', 'DelhiPolice', 's_palani', 'SujauAhmed', 'iyersaishwarya', 'sandeepvarier', 'A_NabiKhoso', 'MALAVIKAAVINASH', 'riyanwahyudi', 'mondodari', 'jfkjohan', 'labiwa_09', 'BabaJogeshwari', 'pun8sharma', 'FriehaAltaf', 'IndianYash', 'anantgoenka', 'AcehDinsos', 'snehiil', 'SandipJoshiNGP', 'ayeshasyahira', 'AsadRashid08', 'Meenu_71', '007ismailkhan', 'Ra_Avishek', 'Memuna_Khattak', 'projectmumbai1', 'ammarmasood3', 'malikaliabbas', 'janakraj13', 'cutejimmyy', 'mahendra3', 'HiraliF', 'SamarSpeaks', 'Muloongo', 'TunstallAsc', 'ashu7077', 'basoya_sumit', 'Awarahaurat', 'Rheumatologistt', 'HuShameem', 'yoshivness', 'IamAnitaKundu', 'Nottie35048951', 'DrRiyamia', 'FastNewsDelhi', 'APSingh1947', 'patelmona9', 'IndieGem', 'galih21', 'duttabhishek', 'AapKaTanu', 'FaridAbrazFA', 'HealthwireMedia', 'aderitonga12', 'amardeepkaushal', 'WuAnping', 'samriddhirai13', 'Sangram_Sanjeet', 'pu_rahmat', 'its_me_chutty', 'ShubhankarKuma7', 'mra_vinod', 'ISO_PAK', 'MushtaqAnsari80', 'mhfalasi', 'mehulved', 'ledygarga', 'urstrulyparvezz', 'itsV_NAYAK', 'EvolveLeadLove', 'RuhilJunil', 'KingMcgregorFC', 'RajeshChakram', 'KotaNeelima', 'haseensarwar', 'NPGohokar', 'Imranbold', 'viraj_sheth', 'leofsaldanha', 'vargheseKgeorge', 'NMenonRao', 'righty108', 'CintaRasulAllah', 'javedSiddiqui0', 'ChandanV07', 'imbrathod', 'SandhyaPanchal_', 'smylingassasin', 'JamiiForums', 'Radhika_Khera', 'ametbakaq', 'UnitedRda', 'AusHCMalaysia', 'Rizvinator', 'raghu_madgula', 'AazadNagrik', 'RM_Tribal', 'snehamohandoss', 'Aryaniyc', 'ShamikaRavi', 'AsmaRiaz09', 'mujtabaamjad36', 'ahmedafaal', 'NeerajZimba', 'janvigangwani', 'alibomaye', 'katranjeet', 'dannicozy', 'CricSuperFan', 'RamMNK', 'NamCinema', 'SaniaNishtar', 'bhargav_gopal', 'jaibhardwaj09', 'RmBkash', 'shenalifernando', 'kavitharao', 'kashikirai', 'Neerajmishra', 'GrandMuftiIsb', 'mahtab272', 'Rajni_FC', 'Jasra_G', 'dineshakula', 'arifazhan_', 'ShakirAbbasi22', 'nilaydutta', 'Jaykaul', 'ashuias81', 'sanjayawasthi01', 'PunjabGovtIndia', 'atishvshinde', 'MaheshSurwase17', 'Neetu_Chandra', 'Orangppinggiran', 'Twt_ParitBuntar', 'nbharkar', 'KanoongoPriyank', 'syedKashaf95', 'AdduShareef', 'Deepak_Ramaiah', 'qaisernawab098', 'AapKiGutthi', 'KhanSaumitra', 'athirisoru', 'DrDeoraj', 'GarimaV80185536', 'VarmaNancy', 'BapatNikhil', 'KamranUmarK', 'MohitKPandey', 'kamalkumar14891', 'udayfoundation', 'MianKaramatAli', 'SFI_CEC', 'soundar4uall', 'NeetuGarg6', 'scarysouthpaw', 'wanmohdikram1', 'Chennaigaga', 'arunkumar783', 'VgforIndia', 'MahimaShastri', 'arun9800', 'NellaiSuryaTeam', 'yatendra_mjpru', 'NancyCNBC', 'zatyishere', 'SameerM_', 'Dippy_S', 'UmerPPP', 'muhammadzakir__', 'BetiMSGKi51', 'Online_khabar', 'PressTV', 'ANI', 'htTweets', 'AlArabiya_Eng', 'siasatpk', 'Hispantv', 'Itsara_Th', 'iamharunkhan', 'Ajay_Bagga', 'FirstSquawk', 'bitaKd', 'the_hindu', 'Rania_bn', 'BlueWalkPoconos', 'thewirehindi', 'Malawi2014', 'Friends_of_HRDC', 'ganeshmantra', 'OpIndia_com', 'RwpPolice', 'omar_quraishi', 'Syedees', 'Asifbhamlaa', 'isseyali', 'rocorpk', 'Mr_Siddharth', 'airnewsalerts', 'CaDalmia', 'AJENews', 'NixinPower', 'KLGLASS2', 'samarinamkhan', 'waqas_x', 'ArunDeshpande20', 'Mojahedineng', 'raedalfulaij', 'AdvaitaKala', 'AjitKDoval_FAN', 'typicalnawazish', 'gaab75', 'ChannelNewsAsia', 'IndianExpress', 'republic', 'MEK_Iran', 'YolandaBulirani', 'MinhazMerchant', 'DrTajikSohail', 'bollywood_life', 'punjabkesari', 'alfinamiruddin', 'etribune', 'AskAnshul', 'nagma_morarji', 'MushtaqSarki', 'arynewsud', 'naeemzarrar', 'jam_kamal', 'janhavee', 'imfederation', 'Shahrukh2526', 'CNNnews18', 'Ch_ManzoorAhmed', 'RakeshSinha01', 'DevelopmentPk', 'sneheshphilip', 'BickyOfficial', 'DevprakashIRS', 'IndoPac_Info', 'RitikaaChaudhry', 'MustafaBaloch_', 'Rg03Goel', 'bijnorpolice', 'Vidyaspeaks', 'Kusum_kesariya', 'VaruKrutika', 'Palaksharmanews', 'thatsTamil', 'Dr_AyeshaNavid', 'AjmatV', 'nidkirm', 'svaradarajan', 'ZiaKhanqta', 'Muskank30286787', 'iabhi_sinha', 'ichiragrajput', 'nazia__jabeen', 'MumbaiPolice', 'sweetnyasa', 'madhujanjagran', 'srinagaradmin', 'Lala_The_Don', 'dograjournalist', 'Mr_LoLwa', 'Madhu7777', 'MirMAKOfficial', 'RightistSingh', 'Ashishtiwari455', 'Brand_Atul', 'onlyyoontv', 'LiaquatShahwani', 'ReallySwara', 'ABPNews', 'ndtv', 'livemint', 'TonkPolice_', 'ARYNEWSOFFICIAL', 'TribalArmy', 'sayedzbukhari', 'RitaG74', 'TelanganaMaata', 'subbiah_doctor', 'SocialKingdom1', 'amjedmbt', 'Amitsen_TNIE', 'mobasshirattari', 'PlantDietitian', 'SMHoaxSlayer', 'BrahmaandKiMaa', 'MahaDGIPR', 'JagranNews', 'Vijay66FilmOff', 'Navimumpolice', 'WIONews', 'DubaiPoliceHQ', 'ind_rajat92', 'narendramodi177', 'louisamsiska', 'Defence_360', 'anuragteddy', 'PoliceMv', 'LoyalSalmanFan1', 'islahmufti', 'RaipurPoliceCG', 'ThaneCityPolice', 'Vadcitypolice', '50o55', 'Yda81', 'Interceptors', 'NavroopSingh_', 'timesofindia', 'singhvarun', 'prafullaketkar', 'Iam__Rashmi', 'uaegov', 'KP_Police1', 'micnewdelhi', 'imMAK02', 'sivasudalai7000', 'filmfare', 'news24tvchannel', 'noidapolice', 'RJMeenakshi1', 'ThanthiTV', 'pranavmahajan', 'mpandit19', '_KIIR_', 'sangramsatpath9', 'TabeenahAnjum', 'gulf_news', 'Worthwhilefigh1', 'SaudiDQ', 'khaleejtimes', 'TODAYonline', 'hydcitypolice', 'DDNewslive', 'KiranKS', 'ipspankajnain', 'samirsinh189', 'Gulalai_Ismail', 'anjali_damania', 'CP_PANCHKULA', 'AhmedabadPolice', 'SChoudharyIPS', 'lakshmianand96', 'gabbar911', 'DrMubarak_Mbbs', '1stIndiaNews', 'DgpPradesh', 'AiliaZehra', 'NavbharatTimes', 'swayamsevak23', 'pratapgarhpol', 'UnSubtleDesi', 'ShumbaGong', 'pss1987', 'DunyaNews', 'manakgupta', 'heismanojmkumar', 'ckknaturals', 'JammuKashmirNow', 'ablmir2', 'AliRazaTweets', 'nayanikaaa', 'saagar_anand', 'REDBOXINDIA', 'SandeepBajrangi', 'snehil_gautam', 'Jana_Shah', 'AjitPawarSpeaks', 'Yashu2374', 'ImRanjeetRay', 'gyanjarahatke', 'PTTVOnlineNews', 'mannkibaat', '8149Zoheb', 'GSVKapoor', 'abhinavtiwari36', 'PiyushSingh083', 'RicKY_KCh', 'fadhelyusyf', 'IYCDAMAN', 'sparjaga', 'ArunBajpai', 'JatNationalist', 'NST_Online', 'go4avinash', 'HWNewsEnglish', 'dhuvasmv', 'jupiter_astro', 'CMOGuj', 'Samiullah_inc', 'sifchandigarh', 'dawn_com', 'kparveen2005', 'ArtKrafter', 'balrampurpolice', 'DgpChdPolice', 'SureshNakhua', 'sumalathaA', 'Narendr43774367', 'Ghulam_qadir_', 'haifa_aishath', 'sagarikaghose', 'Asking1Chatterj', 'jlxmi', 'iMac_too', 'bharianmy', 'Karan4BJP', 'SyedAhmedAliER', 'zarak_khaan', 'M_Hameedh', 'president_uz', 'BloombergQuint', 'IYCWestBengal', 'TimesNow', 'Sasicasio', 'kishanreddybjp', 'NationFirst0006', 'BishtLokinder', 'baraju_SuperHit', 'JurnoPrincess', 'manish_sisodiya', 'HPCL', 'Eatala_Rajender', 'prabinkp', 'Fukkard', 'itisprashanth', 'IshitaBhatiaTOI', 'Gurpreetbtimann', 'Mihaarunews', 'RajivKrishnaS', 'Vijaykarthikeyn', 'IshuSahu01', 'AftabSherpao', 'ifee435', 'rupeshsaigal', 'PTI_News', 'HiremathTnie', 'DostKhan_Jammu', 'DenaBahrin', 'TheAmitLakhani', 'Alekya_k_', 'SuvalagnaC', 'kamaalrkhan', 'PoliceShimla', 'effacing_richa', 'etawahpolice', 'SwedeninPK', 'tfipost', 'girirajsinghbjp', 'Xs2partner', 'prashanti14', 'odisha_police', 'mlaAminPatel', 'baxirahul', '99TVTelugu', 'IPS_Association', 'p_sahibsingh', 'TS_SinghDeo', 'News18TamilNadu', 'RohitashvG', 'NagaurPolice', 'Saurabhk4u', 'dimapurpolice', 'pk92110', 'IGWomenSafety', 'INCMinority', 'bulandshahrpol', 'rkumarmnm', 'Rao_InderjitS', 'Bbygxrllll', 'SEACoronavirus', 'Chiranjeev_INC', 'Abhishek_Mshra', 'Pardeep1925', 'ZeeNewsEnglish', 'vineet_24', 'BlrCityPolice', 'indiatvnews', 'DilipSaikia4Bjp', 'pradip103', 'indianswithINC', 'suilee', 'RajabKhaskheli', 'BangkokPostNews', 'TariqueBuledi', 'abhijitmajumder', 'chriskc_Lee', 'malaysiakini', 'Geeta_Mohan', 'hssn_ismail', 'AliAlvi_Alfaaz', 'Rajput_Ramesh', 'javeednusrat', 'arivinraj_turu', 'UdayMahurkar', 'AdrillesRJorge', 'FunMauji', 'teamsrkpune', 'nakkheeranweb', 'iAstroNOT', 'iSKsCombat_', 'Furqan_LFC', 'Zebaism', 'MoeedNj', 'Dayweekaa', 'iPraveendixit', 'AaronMathewINC', 'drjahanpur', 'ShahNawazKhanMM', 'WokeMuslims', 'iabhinavKhare', 'DevangVDave', 'vinhlhq2015', 'HTMumbai', 'AshishMerkhed', 'abdihaile1', 'ajju_tweets', 'qaem_mehdi', 'Rohith_Mouni', 'javedatahir', 'Dipankar_cpiml', 'ndtvindia', 'tws_pk', 'aajtak', 'nazmaaman', 'rightwingchora', 'MatruBhakt', 'TheDeshBhakt', 'HamidMirPAK', 'madhutrehan', 'latestly', 'dribrahimyasir', 'Ra_Bies', 'Sai_Viswateja', 'jaspreetsays', 'nausheenyusuf', 'ASg1956', 'Shweta_NN', 'SinghNavdeep', 'parthpunter', 'mayur_jha', 'KTRTRS', 'SnapPakistan', 'murtazasolangi', '007AliSohrab', 'pushpendrakum', 'MonaAlarbash', 'MrsGandhi', 'Supriya23bh', 'MothershipSG', 'sanjayips89', 'maryashakil', 'coprajaneesh', 'hassan_k82', 'ChaudharyST07', 'vijayshekhar', 'hchawlah', 'polimernews', 'paranjoygt', 'assampolice', 'RanjanSukesh', 'LtGenGurmit', 'Nagesh_VN', 'friendsofrss', 'QuintHindi', 'MazdoorSangh', 'Alhazzani_Amal', 'MiskwahyaAgwamo', 'Faltu_username', 'haahealth', 'SwarajyaMag', 'AmritadTOI', 'SundarrajanG', 'Saraswatik4', 'yogibabu_offl', 'sardesairajdeep', 'DCPSEastDelhi', 'AQUIBMIR7', 'RLR_BTM', 'thecaravanindia', 'Shoaibsayss', 'TBO_Updates', 'digitaldutta', 'dm_ghaziabad', 'iumlofficial', 'PunjabPoliceInd', 'FinancialXpress', 'labeedaliya', 'sanjaypugalia', 'ArcofOldcity', 'Mitesh_Engr', 'Jhonkosmik', 'AntiCAA_NPR', 'NirjhariSinha', 'TamilTheHindu', 'Neha_IFS', 'Girishvhp', '0086Mt', 'Scacciavillani', 'kamnagupta92', 'NabiyaKhan11', 'Umeshsaini92', 'Sootradhar', 'kktotlani', 'AmynathShuhudha', 'free_thinker', 'zoo_bear', 'cnm_mv', 'Bhaskar97613053', 'manuparida1', 'Shine_Suja', 'RahulDubeyBABA', 'devsahil005', 'zee24taasnews', 'CP_SuratCity', 'ipsnaithani', 'Masoud_110a', 'salman7khurshid', 'mlkhattar', 'ankitbcet', 'fadhelsulaibi', 'kanishkdutt9', 'kumari_police', 'Luckyali07862', '24Entertain', 'mla_sudhakar', 'THE_WAS9', 'ImRitika45', 'iSanjuktaP', 'TheSkandar', 'ss_suryawanshi', 'DeshGujarat', 'Lava_SK_', 'Rajucute510', 'Corona_Soldier', 'TheOliveKnife', 'TheKeralaPolice', 'ProDefLko', 'MaverickHJ', 'SHussainJillani', 'mak_ynwalfc', 'AkashPa46613630', 'ugtunga', 'aneesrajasp', 'valentinnnn0', 'warispathan', 'PCvishnunadh', 'alsuwaifan', 'ssrajputINC', 'AbhishBanerj', 'DeepakPathakDU', 'MnshaP', 'aryaabhisheksi1', 'kaju__katli', 'bbcnewstelugu', 'Shashwa63146733', 'Rahulsipligunj', 'virajsubhramnym', 'ghaziabadpolice', 'SarangiSudhansu', 'IRCTCofficial', 'SMKYF1', 'sapnachoube', 'arunpudur', 'RahulMishra_60', 'spjajpur', 'jitengajaria', 'naveenyadav__', 'prettypadmaja', 'LiveLawIndia', 'jaipur_police', 'knowthenation', 'AlRiyadh', 'HarishBalchand1', 'Bollywoodtrack_', 'AJAYAWASTHI108', 'news7tamil', 'HanuNews', 'VarierSangitha', 'ashokepandit', 'RayMiss_Tanwar', '_Leo_Girl', 'mahendra1186', 'PilmoreBedford', 'amitanatverlal', 'nimmaupendra', 'TOIMumbai', 'Fairoz_JK', 'RaviTiw99881685', 'MadrasMobile', 'NoSylviaPlath', 'swarup1972', 'sheela2010', 'harshkkapoor', 'Suphyanjr', 'mukulfaiz', 'ProfKapilKumar', 'HarshaKsn', 'KapilMishra_IND', 'dmgwalior', 'nehaa_sinha', 'dt_next', 'mrmukarjee', 'OfficialDPRPP', 'obscured_minion', 'JattVicke', 'Adv_RKRanjan', 'ajitdatta', 'Devoleena_23', 'hemirdesai', 'DGPOdisha', 'DRDSDHILLON', 'Bodour', 'dr_maheshsharma', 'MBQasmi', 'DaaruBaazMehta', 'RiyadSameer', 'sri50', 'Jagat___Janani', 'sonalikrishna', 'Indemb_Muscat', 'BBCNewsAsia', 'swati_gs', 'sureshgelot1', 'SuvarnaVeerappa', 'boomlive_in', 'News18lokmat', 'BollywoodlifeHi', 'CoreenaSuares2', 'SwatiKJain', 'HuzaifaSaiyed', 'Arvindharmapuri', 'KiritSomaiya', 'abuasimazmi', 'ArnabGoswame', 'TarunSouradeep', 'krishna0302', 'collectorvalsad', 'AHPOTNIS', 'AslamShaikh_MLA', 'industry_hit', 'mesunainah', 'Maha_MEDD', 'nashikpolice', 'firstpost', 'PuneCityPolice', 'AfrozShah1', 'ShefVaidya', 'Navayan', 'abpmajhatv', 'CAChirag', 'RishikaSadam', 'anusha_puppala', 'DGPMaharashtra', 'DPVEU_', 'MumbaiMirror', 'zafarsareshwala', 'rajdixitindia', 'sonalchauhan7', 'Spotboye', 'SQMasood', 'PCcityPolice', 'CPMumbaiPolice', 'EbaratMumtaz', '01_choudhary', 'taran_adarsh', 'agasthyakantu', 'lafamilleadams', 'sagarajput24', 'NiteshNRane', 'MrccNorth', 'pdeshmukhv', 'muglikar_', 'suchitrak', 'Sumiran_KVS', 'arvindkumar_ias', 'SpSolapurRural', 'gopugoswami', 'AkanchaS', 'jaywankhadejrw', 'iamchalasani', 'TeamTarakTrust', 'gvkreddi', 'telugufilmnagar', 'abhayojha', 'shreyasgroup', 'Subhash_Desai', 'SVC_official', 'puneruralpolice', 'KTR_News', 'saurabhraajjain', 'SumitraRoyTOI', 'frustratdINDIAN', 'Info_Pune', 'Pishachadhiraj', 'DrNausheenKhan', 'Ravi5308', 'LoksattaLive', 'pravesh814', 'katul67', 'Moneylifers', 'dennismarcus', 'imvivekgupta', 'ChaiMIRROR', 'sunita_krishnan', 'anandmahindra', 'kuncheraghu', 'rekha_9999', 'PetaIndia', 'aparanjape', 'Manasi71', 'THubHyd', 'jaavedjaaferi', 'KChiruTweets', 'RatanSharda55', 'supriya_sule', 'netshrink', 'keshavupadhye', 'VjayDeverakonda', 'AEHarshada', 'sundeepshawarma', 'Avinash90686412', 'meeracomposes', 'Pravsw', 'furkanaibani', 'mybmc', 'sidharthfanfrve', 'mohol_murlidhar', 'suniltingre', 'SidShirole', 'ProfFMustafa', 'AliKaashifKhan', 'moneycontrolcom', 'Pune24x7', 'deshdoot', 'iimcomic', 'mayur3c2', 'Rowdyfan_Nani', 'HEROVIJAY9', 'Jadhavdeep1', 'Vidyaraj51', 'Habeebinamdar', 'RohanRChawan', 'AnaghaAcharya', 'Praviinjaiswal', 'withSNirupam', 'JAGANTRS', 'SKNonline', 'LavekarBharati', 'bhogleharsha', 'RatnagiriPolice', 'RowdyFan_Manu', 'SitaraCinema', 'HiHyderabad', 'shobegumpet', 'filmervishal', 'revathitweets', 'NtvTeluguLive', 'KajalTrends', 'ParasharMone', 'Rajivgupta1515', 'asmatasleem11', 'vamsikaka', 'OfficialMSGfc', 'AIMIMZuberMemon', 'Kuldipshah92', 'mohitbharatiya_', 'Sadafsam96', 'KillBillBride', 'kochattil', 'TV9Marathi', 'RamCharanCult27', 'RDATT_', 'Collector_MDL', 'AhirsachinAhir', 'onlymaheshfans', 'ViditaVaidya', 'diptivsingh', 'AparBharat', 'AISFMaharashtra', 'JSPVeeraMahila', 'VallabhMIRROR', 'SirPareshRawal', 'murtuzatweets', 'ravindra__mr', 'SunnyLeone', 'zoru75', 'ayushmannk', 'mgnayak5', 'DoctorNene', 'fans_hyderabad', 'Rahulmishra00', 'SudheenKulkarni', 'sambhavnaseth', 'AsliDevil', 'DrGaneshCartoon', 'Babafasiuddin', 'bb_thorat', 'pacckup', 'INCMumbai', 'PinkvillaTelly', '8pmGyan', 'VijayGopal_', 'pratyasharath', 'PravinKalaisel2', 'glamsham', 'KumarReddyCH', 'sanjayezhava', 'AsaliPrakash', 'hsachdev', 'Gowtam4n4', 'nagaonpolice', 'nisheethsharan', 'insane_suraj', 'TelanganaCMO', 'praful_patel', 'vishy_vishal', 'vineetjaintimes', 'ashishchauhan', 'LaturPolice', 'IamManasiS', 'mid_day', 'iAmitRSharma', 'SrBachchan', 'WesternRly', 'sandipGopani_', 'anirudhganu', 'alankar6427', 'mataonline', 'suchetadalal', 'sahil11p', 'kagtireema', 'sonudub47749189', 'anvivud', 'RazaOldcity', 'ActivistTeja', 'Fatima_Z0hra', 'R_A_J_P_U_T_', 'Telangana_360', 'ByRakeshSimha', 'ChaitraSalian', 'Bajaj_Digital', 'pandeyyJi', 'saurabhv99', 'SeethaRmMallela', 'archu243', 'collectorgirsom', 'sahiljoshii', 'sanjaynirupam', '4vinod', 'jyothsna_tdp', 'rameshmashelkar', 'NavedJafri_BOO', 'mathurjony2020', 'rock_lungleng28', 'GossipGanj', 'THHyderabad', 'Arihant17619159', 'CAPratikKarpe', 'InfoDhule', 'Sumra_tweets', 'AnilSinghvi_', 'iAbhinayD', 'mumbaitak', 'AditiIndiaFirst', 'mask0000007', 'ShaliniShah20', 'Raja_Africa', 'Akshay__R_K', 'ravithinkz', 'pinkvilla', 'Shubham30409566', 'teamTSIC', 'satejp', 'boxofficeindia', 'Mumbaikhabar9', 'MeerraChopra', 'iamkandula', 'yschowdary', 'bollybubble', 'tarasharmasaluj', 'SaipranayC', 'vipinrocs', 'DrAzra143', 'Shruti_Nagpal01', 'PMCPune', 'velumania', 'Vny95130855', 'jaimahakal555', 'sandeepkishore_', 'gattanivijay', 'aditya_madhava', 'ShivHindu879', 'imabhi0404', 'divyenndu', 'ameytirodkar', 'JaiHIndtweets', 'samirgurav', 'nawabmalikncp', 'ABVP_MU', 'mritunjay118', 'NeerajCKumar', 'RanaAyyub', 'indiatvrajiv', 'Joydas', 'VPra52', 'ABVPVoice', 'adtpatil', 'Nikki_Fierce22', 'ashishnavandar', 'anilthakraney', 'Varada_M', 'OMKAR_BORKHADE_', 'AbhiSin76206686', 'ABVPMumbai', 'milindsmarathe', 'ABVPKonkan', 'Paul_Oommen', 'CRPaatil', 'Parbhani_Police', 'RanjitSavarkar', 'MiLOKMAT', 'GayatriPYadav', 'docraviw', 'Nigel__DSouza', 'snshriraj', 'TruthRoars', 'Sanju_Verma_', 'Kalpesh64762640', 'vishal15_4', 'vbsingh60', 'SandeepDhonde8', 'balramom1', 'ShriManoj9', 'Malay3262', 'urstrulyBMH', '499Undertaker', 'Manasi73', 'AmeetSatam', 'arunv2808', 'TheBombayBombil', 'Chopdasaab', 'Vishu_CS', 'StuteeMishra', 'vijaysghule', 'boomzy1231', 'shailesh505', 'SujayRane', 'sohitmishra99', 'mangeshspa', '0mNam0', 'RoshanJ49403678', 'PJkanojia', 'ajhay9', 'QutubOwaisi', 'the_zakirah', 'MishraRambahor', 'AshokChavanINC', 'satyajeettambe', 'sbajpai2806', 'pramila2710', 'aniket_survase', 'hemalkanani09', 'NashikOnWeb', 'uddhavthackeray', 'masumhassancho1', 'shweta_shalini', 'KarunaGopal1', 'DEVenDrapai', 'avinashkhatik2', 'AerpulaVenkata', 'gurichopra', 'thisisindra', 'ChitraKWagh', 'sandipsabharwal', 'JoshPune', 'ssvirendra', 'JMehta65', 'swastika24', 'RanvirShorey', 'AlwaysRoopeshC', 'TrendsAlluArjun', 'AIPCMaha', 'bgopu1973', 'BKiran_1995', 'MazahirRahim', 'BYS_Mumbai', 'V6_Suresh', 'prachee_mirror', 'dk_mahendra', 'Sanket_News24', 'mk_mohitkhurana', 'kunalchordia1', 'msuhaib_raza', 'Soumyadipta', 'KanimozhiDMK', 'Waseem_Ahmed11', 'Shankaracarya36', 'im_saiganesh', 'jayapluschannel', 'INCPuducherry', 'sunnewstamil', 'VanniArasu_VCK', 'igtamil', 'LMKMovieManiac', 'filmibeat', 'Jay_Apoorva18', 'XpressCinema', 'PTTVEnglish', 'thalaivarTN', 'NairShilpa1308', 'mahajournalist', 'adhicutting', 'CMofKarnataka', 'anugula', 'NewIndianXpress', 'sgurumurthy', 'shivakumar_tnie', 'rajeev_mp', 'petleepeter', 'tamil_twtz', 'vp_offl', 'JPtamilnadu', 'Stalin__SP', 'imravee', 'Brijitdighe', 'swasthi_cs77', 'DrParameshwara', 'PMadhwaraj', 'nimmasuresh', 'TheAhmedShariff', 'D_Roopa_IPS', 'AIADMKITWINGOFL', 'rhevanth95', 'VanathiBJP', 'BPCRetailMysore', 'megirish2001', 'srinivasiyc', 'VNarayanasami', 'StayingReal0511', 'dhanyarajendran', 'TVM_OTFC', 'dpkBopanna', 'arulgreen', 'prasannavishy', 'bst_arya', 'Mahi54077638', 'smitha_tk', 'oneindiaHindi', 'bigfundu', 'jsamdaniel', 'hemanth_kgowda', 'prudentgoa', 'BangaloreMirror', 'publictvnews', 'RangarajPandeyR', 'isitso15', 'avprameshbabu', 'HMLokesh', 'RKRadhakrishn', 'vschanna', 'pram050322', 'samar11', 'niharika_bhatt', 'BTP_Offl', 'PunjabPMC', 'cmseth1', 'NITIAayog', 'RusEmbIndia', 'eOrganiser', 'kuljeetschahal', 'AunindyoC', 'advocate_alakh', 'DilliDurAst', 'TweetwithAman_', 'Yuvraj721', 'drsfaizanahmad', 'kawalpreetdu', 'sanghagurpreet', '_YogendraYadav', 'NH_India', 'aligarhpolice', 'ManishS80590326', 'JThakers', 'Ahmed1Hilal', 'RajatSharmaLive', 'aarifshaah', '__doraemon11__', 'MissionAmbedkar', 'khan_mumyyaza', 'delhidefence', 'gorakhpurpolice', 'mygovhimachal', 'rawnksood', 'ashwani_mahajan', 'suraj_yadav2005', 'saket71', 'DrBKKundu', 'BankerNavyWala', 'NBTDilli', 'BJP4Delhi', 'LaxmiBi87208189', 'AyshaRenna', 'kgahlot', 'hafizgandhi', 'teejaybaba', 'alasaarela', 'BucharestStray', 'PaiviAnttikoski', 'JackVardan', 'zorancicak', 'DraganGulic', 'dekker', 'stonexman', 'VladiBabic', 'villecantell', 'RealBobbyDino', 'JussiLahde', 'CanicDuca', 'MilanCHE111', 'FetishNatalia', 'EstonianSaunas', 'seljancura', 'HeliVuori', 'jlehtoma', 'rikukeskirauska', 'Frank_Supercell', 'radicaldaily', 'twisty1145', 'TheAngryLord', 'BELIBOG1', 'MarkkuPeltonen', 'PonziDemocracy', 'DmitriTrenin', 'velvetxears', 'Ermowhat', 'vojislavostojic', 'ylenews', 'Gabbar0099', '5tv', 'helsinkipride', 'DogsareLoyal1s', 'manuderetti', 'politic_sass', 'BlagicaKostic', 'THLorg', 'BoyerAna', 'OBrother_op', 'TaaviRoivas', 'HNohynek', 'Ulkoministerio', 'HelsinkiPoliisi', 'daksinvrindavan', 'PToveri', 'Wonkette', 'IcanKudivo', 'ItsRachelDobbs', 'Fl0r_Geneva', 'CINSerbia', 'geoduque', 'debi_nr', 'srdjandrago', 'savarnafascist', 'MariaOhisalo', 'wanlov', 'VchCientifico', 'Chuletti2013', 'AeronewsGlobal', 'joonaslinkola', 'Ansip_EU', 'markodjuric', 'Dusan07', 'Viv_Dany', 'Djak0vic', 'BlokkiMedia', 'Deshysmalls', 'Maa96507870', 'Heino1Olli', 'ashtyn_drake', 'Kornelij', 'NeilPHauer', 'LinkedHD', 'mithisa_motho', 'Br4ndonyo', 'ElizaMajumder', 'RoopaSpeaks', 'champagn3carti', 'mrunalinni', 'Pakistaninpics', 'DrSJaishankar', 'MandiraShastri', 'sandesh_samant', 'Ksingh561', 'Chandrakbose', 'ayeralobaid_', 'NagrikSatyagrah', 'shubhankargh', 'opinion_h', 'unicefchief', 'AtchuthMohan143', 'fatyhaxx', 'D5habib', 'RandeepHooda', 'AboShawarma', 'Sukhveer_Sandhu', 'x_Tulip_xx', 'AlluAni13756384', 'KicchaUniverse', 'soniiannup', 'MayukhDuke', 'iwanteggtarts', 'jessy_sai', 'Soumyad85742777', 'theonlyizzah', 'ShafakatK', 'tinobest', 'priyankadeo', 'Khaskhel_Aijaz', 'RealSexyCyborg', 'amilaik', 'KyonKusanagi', 'Mansaysahu7', 'omarfaayiz', 'Blessed_Soul_1', 'ApkaApnaShrenik', 'tasha__koh', 'worldscouting', 'AtulMimpimika', 'mynikkaname', 'KP24', 'KumarSarangapa1', 'seonho_lai', 'asrashok', 'RathanRameshRR1', 'AnilTpyc', 'MansoorHayatLak', 'sssingh21', 'SaifForSRK', 'manasmitak', 'crpf_srinagar', 'Choudharypnc73', 'sharifahwanii', 'TianChengWen', 'ItsAnasNazreel', 'ShivaniPatel23', 'priyanka4nation', 'PencyS', 'RandhawaAli', 'SujitBisoyiTOI', 'shailajateacher', 'satish_gowdru', 'veerakunaparedy', 'dasxtej', 'ArvinderSoin', 'HarshdeepKaur', 'kinggowtham', 'flightofsand', 'Biorahul', 'ChapmanLizzie', 'mhdhafizee', 'Amerny3', 'sezairi', 'SwordsnFlowers', 'sharmashri', 'narottamsahoo', 'witherthorne', 'DarcelAnastasia', 'Ganeshspeaks', 'danielfyork', 'bhav2406', 'annazrulan', 'rnivruti', 'Maryyahh_', 'kumaran4kp', 'Kavita_M57', 'SandipGhose', 'lakibuteka', 'saraalmousawi_', 'kounaljain', 'poojabeditweets', 'rayhanburhan', 'BangaloreAshram', 'neelbs', 'topengski26', 'gyagnesh', 'AliFini', 'adaa1nonly', 'bolozubaankesar', 'graham_euan', 'SanwarYadav', 'SureshKonatham2', 'coachshujau', 'ArfaSays_', 'abhidipn', 'jai205ann', 'keekl7', 'sandeep_potla', 'sibumabena', 'sanjeev_goyal', 'Sweetqismina', 'GERinPAK4youth', 'j___k_m', 'llssheemall', 'Laiba_Zainab', 'Faizans_7', 'thefarahjo', 'DSBcricket', 'YaraQassabb', 'onzXonz', 'uibglobal', 'ramishkad', 'DefeatKorona', 'AadilSRKian', 'nabeelaq', 'Sameeksha_Yoga', 'almxhairi', 'Aishath_Iyadh', 'JaniMemonPPP', 'JoliePagaille', 'infitahnatasha', '_Ramanathan', 'iamchika14', 'Littlemrs_A', 'thatsmyThucents', 'pkhotajspl', 'LicypriyaK', 'vanessa_vash', 'gri_tay', 'shriram_l', 'Dhanesh91783211', 'BhavreenMK', 'a01_g', 'THeISSUeeee', 'DrIqbalENT', 'chalai71', 'bitkashyap', 'SaifulHaszrul', 'itariqanwar', 'the_WadoodAFZAL', 'DarshitJoshi_', 'HarshadeSilvaMP', 'WealthyBhai', 'nirbhaisingh07', 'neuroayush', 'SpandanaHarish', 'mariumsummer', 'vedant_red', 'RahulDevRising', 'sevakaroon', 'Yusfariqq', 'SihaamMumin', 'sahilk', 'AllariSuresh1', 'cnpaulpradeep', 'JairajSinghR', 'pantparitosh', 'afiqdruce', 'EUPHORIAJJK1997', 'ErJayaBharti', 'JamesJGuild', 'MsJoelyFisher', 'nnevvy_19', 'aryfhmzn_', 'jiteshmishra01', 'leilarostami99', 'sanaejaz2', 'AlzainAlm_', 'ShaalV', 'JournoAshutosh', 'RashidMkhan79', 'AtishiAAP', 'JemilahMahmood', 'vikramyadav', 'AnupKulkarni2', 'ashishrsharma', 'ashnazaveri', 'BawalHuMe', 'walabs', 'mshaanshahid', 'lishasya_', 'SebCochard_11', 'h3ntaihunny', 'Iamrahulkanal', 'secretmoments13', 'alankurd1370', 'SumonaChakrabo8', 'pakistanned', 'SheroSobti', 'nrnxzhda', 'sanchihihii', 'AdvSanwar', 'QurbanALaghari', 'im_afiqq', 'RiteshK37156503', 'believe_0369', 'JinnahsRepublic', 'OverLord_Xd', 'DeevaPriya', 'UmairMehboob404', 'SohrabBaluch', 'ArhamSayss', 'Binnyr77', 'Samraa_says', 'THEMARYAM13', 'DrAlsharija', 'Baktawerr', 'makesyoucakes', 'SharjeelUsmani', 'sadafjafar', 'omthanvi', 'alyaiae', 'ebrahimj_', 'FPHassansheikh', 'NoorZRafiq', 'foreignluu', 'BIHARIBALMA_', 'fairy__fk', 'hadouken51', 'ChupShaa', 'MuzzammilAslam3', 'allforsaintsup', 'noormujdded', 'RahulSinhaBJP', 'Ebenmugeez1', 'Samanyanga_', 'IamMaheshFan', 'Sanghasena', 'beingtweetman', 'ProfSomashekhar', 'amberswami', 'trisyiarobaid', 'PTINA247', 'drkctweets', 'rohitkumar9848', 'gauravwankhede', 'kumarmanish9', 'Musthw', 'dharitri', 'Nurfa_nfs', 'lifeofapatrakar', 'izzDaniel_8', 'AdityaChapraMP', 'NepeanSeaRoad', 'shivanipaliwal', 'Zikryirwvn', 'dul_U', 'aisha_adam', 'vkbandhu108', 'nurinafiqaaah', 'ipputts', 'luqy92', 'unimalaya', 'sahibdadkhan', 'bsarwary', 'jamesphillips72', 'Sadiajavedppp', 'GrGuru48', 'Clar191', 'ademisireoluwa', 'Deewalia', 'AmyraDastur93', 'rajeshsawhney', 'JenMusthafa', 'Brajeshsupaul', 'SKPBGopinath', 'guywiththestar', 'RamalanEisa', 'amirmerican_', 'victorshome1', 'adgzoneagra', 'BubblyDentist', 'Fzroy_', 'syedwajahat7861', 'anandfateh', 'temjentoy', 'AroraKewl', 'BPCNagpur', 'DeSatyanara', 'AYP1293', 'Satyaji38', 'BishnoiRav', 'Monam54862763', 'SyedWaseemMNA', 'MRafiqMunna', 'Sauravrajput111', 'yadavramya14', 'Mr_UsmanAnsari', 'AnshahraA', 'alexbhturnbull', 'iKabir_Khan', 'vishalnautamlal', 'AlawiyahYussof', 'milankumar_sam', 'halalnosering', 'AltafBhutto_PSF', 'hanishx', 'ABANGPATIH19', 'vanstable', 'briscoepark', 'casuarineslove', 'GreatGameIndia', 'cyalm', 'deepsealioness', 'PARInetwork', 'SYNCHPakistan', 'faraz_kandhro', 'ImHarisAli', 'SpudLovr', 'ikeeyah_', 'Moonstonekv', 'garimagarg_', 'jinyeoluniverse', 'zakiavelli_v2', 'muhdpc', 'WhyAisha', 'rauhlmeezy', 'djaywalebabu', 'INCTharoorian', 'HanBonBon', 'comradesxm', 'reportervtvijay', 'tolubeats', 'n_fatimah9', 'leedeeyuh', 'elihoole', 'folitically', 'goformatty', 'itsmaryamdaya', 'SristiJhunjhun1', 'aaliaaaliya', 'iSendim_', 'deerao', 'AISA_tweets', 'tk_hype', 'ladywithflaws', 'Arora16Stuti', 'Feedppl', 'mirulroslii', 'Azoozz_', 'SRajumanthena', 'LadyVelvet_HFQ', 'AbrarbiintN', 'ZarrarSalahPTI', 'NicholasK111', 'DrWaheedMiraj', 'the__uncrowned', 'SanaaTauseef', 'patttyjax', 'IamHisAnchor', 'IamHiralRana', 'Prashantvarmaa', 'ActorArjai', 'AtifMateen2', 'AliNHamdani', 'SRKxSuman', 'vcanmakechange', 'nuhasufian', 'bindadey', 'iamLiyaNashita', 'AafoNaaz', 'bxbyunic0rn', 'avenfauzi', 'sufimusafir', 'ClintyCJ', 'JayvijaySachan', 'JonoH', 'rohshah07', 'PYOSindh', '_Unique_Lover', 'diggles_7', 'Jeeveshu', 'Adibleesimmons', 'DivyaSoti', 'DhavalBalai', 'Mavsour', 'm8_4w', 'VenkyPatnaik1', 'YipOhYip', 'ssjtrs', 'MaheshGinugu', 'iPradeepSangwan', 'KingKatriar', 'MohdAsriDungun', 'abdulmalikmv', 'Yajid_', 'asmeeseva', 'DrParulSharma1', 'finemanhocS', 'FahiimaBicir', 'leenadhankhar', 'KazmiWajahat', 'adysyaf_', 'joecharema', 'drew_biebsss', 'TapanSatpthy', 'JeetuJamali', 'HamidiBanoo', '_Bal_Krishna', 'JaEEnY_ChouhaN', 'Thesanjoy', 'paramspeak', 'AidaActually', 'lulwuh', 'tanejamainhoon', 'abswitter', 'hazimlamasia', 'RunNerdZukhairi', 'ScottShuey', 'hvmmvdvrt', 'vishalkapoor91', 'Kumar1986Sandy', 'GajendraPatil18', 'Dihendra_Sahu', 'swaragamafm', 'khalids', 'TotoRahardjo', 'iwanpiliang7', 'kam_227', 'mafeitz', 'moqbil_algofiry', 'WasanYounis', 'Riyadhnews2030', 'edimaha233', 'khaledalburaiki', 'DrBhattashok', 'md_almousa', 'IsmanTengku', 'Nahedalahmad', 'TaghridAlhowish', 'refrizalskb', 'Animmina218', 'khaled_NK85', 'AbdulazizSay', 'sharannya50', 'Ql979', 'AlsoBajwa', 'wanfayhsal', 'Alhitmi79', 'KhalidSH77', 'preeti_chobey', 'm_alwan', 'mmohammadii61', 'jallamki', 'bin_asj', 'meyysi', 'harpalinsan3', 'noralshaman', 'hamad_jassim', 'H_swilhy', 'Nanoo__02', 'A_A_ALNasr', '9Kora', 'vishwajeetku84', 'hh__1111_', 'ah00med3', 'Q400300', 'MuneeraKA', 'faisalbinffahad', 'Reem84837264', 'Imanenahoudy2', 'AshwaqAlAli75', 'SYahyaHussaini', 'Dralbader', 'khkxd', 'Dr_Suadalbesher', 'moedflood', 'Attawat2un', 'ChAwaisGhumman', 'fahedalbishi', 'MSAlmoqbel', 'halkadi', 'raidezar', 'alialgofaily', 'rajeh_90', 'itsasimameer', '_4hem', 'HadramoutCenter', 'greensheikh', 'wafra_coop', 'sulimanalmanah', 'alainalkaabi2', 'yaraibrahimn', 'fwaid_Hadith', 'akhilesh_psingh', 'natachasabeh', 'MUSAAD76', 'm93ssm', 'HussainAlQemzi', 'mahdihumaidan1', 'omr94_', 'NASEER_OB', 'TarotDarma', 'BaatYeHai', 'altamimi14', 'albishi', 'HaaifaSh', 'gh_roshanak', 'saud264', 'OmarAlRajeh', 'Mjathlani', 'faisal15mm', 'ksu_cams', 'rayyan_ad', 'HosseinMahini', 'abdullahashim2', 'FelwaAlmazyad', 'KhalidAlenezi1', 'MaramAbdulaziz', 'ASNA_20', 'Ahmedhakbani', 'Mishaelhum', 'e_manavi', 'OthaimMarkets', 'KiraakSalmaniac', 'otaibikm', 'Pradeep28950055', 'sameer_alnamri', 'SalahSa68159403', 'nojoud_r', 'abuomar2226', 'AAmarulazlee', 'muataz_m7', 'icantikhehehe', 'cst_nesss', '022puneet', 'bepejeel', 'Barabankipolice', 'WafaDekhail', 'itoshimandola', 'ALKAAGARWAL9', 'Khalid7117111', 'ainsyahmie', 'akhilkatyal', 'ahmedelanazy', 'w2P5v9HGVuBXLuW', 'jiladola', 'babybinbaik', 'sys_sa', 'ajitanjum', 'AnneChom', 'mmohammadrezza', 'saNOOKer_nook', 'PerthsaintC', '_garrywalia', 'LemonLe91456250', 'yovitaaapr', '6multilingal', '02Umam', '_bks2828', 'kakarlee_', 'rrr11142', 'lalisawin327', 'BilkisPerween1', 'jameela56_', 'apad611', 'nok_newvoice_', 'gsramaswami', 'RedSkullxxx', 'SosweetYui', 'BbTaehiong', 'hananzz', 'mohamedahou20', 'Thaikorn1', 'Fatima_Gillani4', 'ahmhaitham', 'SyedMuhdFadli', 'sillysampi', 'Shantan89075404', 'setkabgoid', 'manabubannai', 'Pinitvit', 'tkdnxv', 'Azuri99', 'janiaabaniaaaa', 'detikcom', 'GP_6395', 'fedtriyahya', 'akm42561907', 'PreetiS99099323', 'FujiyamaOden', 'Silversay74', 'drkhankpg', '_aamajang', 'ks_ruk', 'sarachodhry', 'Tip_of_the_day7', 'Imroh07', 'viitufii10', 'buddhadasa', 'erna_st', 'Mangal02051993', 'Baby_Patata', 'Aadiiroy', 'HassanAlobaidly', 'ItZzAli_', 'Snehlat27685620', 'Weed0690', 'sonalgoelias', 'nouf_noo0oof31', 'HSamiullahniazi', 'Alshaikh2', 'yiklilly12', 'athkar_aa11', 'SinarOnline', 'MissblovePalit', 'MohanCRaja', 'nenanee', 'Zmx10VLE63LIHrb', 'AlamgirMian', 'AtulgahlotBjp', '6mboora', 'MrVop', 'Tracy13275893', 'PROMOTE_SAINT', 'SAAARMYs', 'faiwateam', 'HendraaPd', 'dinesh_OTFC', 'Anggia78365585', 'mohb_Anesmansor', '2Sh3or', 'alhusaini2406', 'Sadaf_v', 'faizimughal20', 'kemudiaaan', 'FirozSRK555', 'putriteratai25', 'Ahmadaljaralah', 'RedStarrTheo', 'Jannabilah', 'YallappaPs', 'liquidlaugh', 'sakamobi', 'CBA_58', 'Anand_arya246', 'dalipin68', 'KSA08351500', 'SantoshKhaderi', 'tempemendoanGL', 'Pampamo0', 'Supasu_14', 'Mia_Andyy', 'TFC_mass', 'MorningNewsTV3', 'al_jazirah', 'cnnarabic', 'Swamitirthdev', 'ZaheerShykh', 'kangmirdja', 'kimmohito', 'MS_ALRASHED', 'mshidqi11', 'BonadMon', 'queenbsav', 'bilarakib', 'soso_2o21', 'willayati', 'Jayati1609', 'RamdaniYosa', 'shffnaaisy', 'SumairaKhanLive', 'asrulhadiamir', 'BIG_HASS', 'ricky_un1ty', 'Thairath_News', 'Hana_Andrera', 'Felipe_Arco', 'AlshAsaam', 'anasaudi2030', 'Poppyyyz7', 'ohkeyyy_', '501Awani', 'sarasalsaud5', 'IiiArm', 'KhalidSamad', 'wtuntawiroon', 'Roopess', '7ssry', 'aziz1alsheikh', 'gulzarsiddiqu19', 'Samsul60568688', 'KemenBUMN', 'mbuyabis', 'KavyaRachna', 'AlGhurair98', 'arzazizi', 'sabakhan1045', 'echahaie', 'aaaacii', 'afyanIFP', 'PihuShah30', 'RealHimalsh92', 'teabhoky', 'turke8888', 'gmm25news', 'benchamaphorn2', 'Chakletigirl', 'markpeak', 'MissDioroffic', 'AlHadath', 'ReenaS100589', 'ZuhudRijal', 'ThakurShamsher9', 'ForestsMatter', 'Raghad_alsaud', 'noviaaika_', 'ZhanZhanTH1005', 'Sarikansarden__', 'tylcreamsm', 'NananaSrita', 'ShaymaOmar', 'Mahesh10816', 'Manzoor65096707', 'leamaine2019', 'BchHng48178678', 't_alshdayed', 'dalangary', 'ZeirosLion', 'AOLSwamiji', 'ALMRISEUL', 'wachingyou', 'Talitini', 'lovelySalmaniac', 'Agustinekyy', 'im3ax', 'Dharialwazzan', 'BernamaTV', 'theoldtale', '1_childern', 'I_M_333', 'k_izzuddin', 'XXXbyefelicia', 'DHRichardKo1', 'Samar_77', 'natsing5', 'amazingizzue', 'musniumar', 'mtigermemon', 'Deep4IND', 'tapiiqbaalqibul', 'Zakaria111214', 'NARENDR20249946', 'i2_l1', 'Ahmed_AlBreki', 'ftjsn', 'RishiPrasadOrg', 'Kavin_kannann', 'KemenkeuRI', 'almulla_a90', 'guangne', 'Live_Hindustan', 'HoerulRiski', 'FadzilahMamat', 'P17900', 'tangkwaja_kha', 'buntyinsan74', 'noon_htp', 'palga_pe', 'MolonlabeBerni1', 'kiebieart', 'l0fam_', 'imranmalikst', 'stdsiliwangi', 'iyaatauhasss', 'Allinz_', 'ZhuzhengtingT', 'ThaiPBSNews', 'asrulmm', 'salrashed', 'yudhazulffan', 'CNBCArabia', 'toynashana1', 'PinkPandaClubTH', 'officialAtinSG', 'ihalimasrar', 'bundarajakhalis', 'UnIxphysco', 'tsaniindaeyo', 'Real_Sihan', 'alvinhuhu_', 'MisiahTaib', 'fawwim', 'nasarghouri1', 'OVOarchive', 'jentikhati', 'abelquaety', 'iamfarooquiii', 'cupakai4', 'Wriizzyarts', 'LesIieLopez_', 'AshrafAriff', 'Puri_Official', 'noookkiee', 'Shifaa_Lilsodor', 'MrNaseerKhan04', 'bax_r2', '_larauai', 'Ro3ah23', 'Mr_3azmi7', 'madamau4a', 'bassam13_1973', 'DrChungYC', 'Basantp54857331', 'AboutTNG', 'DaddyShiva', 'Maaz__Yo', 'jakpost', 'Air_Ja_Gutz', 'FuzailFaiz', 'AshutoshKrUP1', 'waswasqahri', 'GLZooJogja', 'Bollynews_2020', 'friyda__', 'dickymaulidy16', 'baneenrizviii', 'aa335aaa', 'teooaswaraa', 'gift_zuuzaa', 'a7madHadi', 'OshoPrabhakar', 'alaynarosaxo', 'SultanHabadi', 'm7mdMQI', 'SuckaFree_Nell', 'msirsiwal', 'ThaiPBS', 'SN300S', 'Ibrahim_opm', 'SomaliPM', 'firasd', 'muslimorid', 'raishussin', 'shekhar_mande', 'Bilson_Rai', 'EHdhanhani', 'eko_kuntadhi', 'bedugcirebon', 'Kavinthans', 'gandhimohanr12', 'Erode_Movies', 'RajendraGhunaw1', 'SurajThakurINC', 'HashTagFirullah', 'Andhra_Sena', 'AnandTarock', 'abuthary361', 'BNT_KHALEED80', 'ewin8923', 'omarabdr_', 'NAlnasr', 'zEsHan_xafar', 'drwaque', 'alazmeee', 'the_s3eed', 'KaleemU29906599', 'AmandaCerny', 'mahditabar', 'PitAbrhm', 'chadnan751', 'paramdisciples', 'mansour_daban', 'sapiksam_', 'noorzaidirohma2', 'DokterAri', 'MAJEDLFA', 'charoenpura', 'maragov', 'booedi', 'aldrai3i', 'Myien', 'SSolouti', 'h_s_almazrooei', 'amir_newking', 'Deyab_AlKhamiss', 'Teemo95', 'mansoormansay', 'ibahzad', 'alhilal_q877', 'OsamaAlMunawer', 'SBushraIqbal', '_tvx7', 'b_4vb', 'mansorsaleh1971', 'Zezo_rmd', 'ATrapenard', 'D_alghaithi', 'arifinipinnn', '__CocOmo_', 'RafaelPoulain', 'Dyyalla', 'mixx2535', 'Bevzlovekn25', 'Al7lhh223', 'iwanjanuarcom', 'kentoslife', 'Mehroahmed1', 'vagxii', 'AdvocatejJyoti', 'yayaqma', 'yhyh122', 'ivancarlooos', 'Marwanalsarkal', 'a7medi794', 'COLUMNISTABBASI', 'Lugkaaaawa', 'alanazifaleh', 'RiznaZareer', 'h_a_h_m_a', 'moonsa7908', '_abeid', 'sehjra59', 'AliKuemkh', 'wedwedwed', 'oxBeerxo', 'picxa_40', 'alshalahi_q73', 'Ahmed_Eyeman', 'dromaralofi', 'Preemen2', 'aniqnaji', 'gabriella_dilan', 'najemwk1', 'AmirpoorMehdi', 'ammadeyimon', 'Alyazyakm', 'Aligh_Official', 'BAH_Int_Circuit', 'ESHDMM', 'pongkool', 'maismmm1', 'rashidalmethen', 'ib_bun', 'prasuk10', 'Saeed_AlHamad', 'hmg_118', 'Dalalhammadi', 'SavitaNishad15', 'ForPalit_Only', 'fcukeveryname', 'Doyok999', 'SaksajeeB', 'kimtititienMo', 'SN_69690', 'MHaffizJ', 'tikky_jy', 'AzharAndiTahir', 'hamza_alkhayyat', 'lidbahaweres', 'y_k_alrashid', 'abusulayman', 'AHMED64AHM', 'faisalalbass', 'sam55th', 'kanxikali66', 'noarah__2', 'dananni1', 'abuali3003', 'M_algnnam', 'DabushKamarudin', 'sehr_rimsha', 'chippy_nonstop', 'farhanmeer', 'MbdInfo', 'ealaf_al', 'AliAlmohsen1', '_amnaalb_', 'salman_aloqayli', 'nikiayushi', 'bomeshall', 'tzghauri', 'Kaddamry', 'LikeMamaDoes', 'AmirSaeedAbbasi', 'Mmukhaini_TV86', 'Deepkaran_INC', 'S_altemyat', 'bts_pk_galaxy1', 'm_tabook', 'bluesherbet_', 'Lawyer_Intesaar', 'AbhaChaudharyUP', 'hamad_albraidi', 'TwinsHadban', 'AmerAlQa7tani', 'Adnyaswari_11', 'banggaber', 'HASAKHAN66', 'khalilfarran', 'abdullahomar22', 'Asthakaushik05', 'lawaktahapdewa', 'IamAsimRiaz07', 'Lahootiii', 'fa9923', 'saber_alshehabi', 'NahlaaSarhan', 'tasshiraa', 'darsono345', 'saifudheenup', 'JAMALALAKKAD', 'DrUditaTyagi', 'Momanij', 'ReemBusati', '1D4N15H', 'ecosocrights', 'LnmeenaSamleti', 'Kadalkannan2', 'mukhtiaralimir', 'iak00l', 'lucky_swapnil', 'nazeelshafeeque', 'NikShariff92', 'taher_aljunaibi', 'jaldoseri71', 'Harshupandya', '_bddran', 'ajay29manas', 'Anant_bjym', 'raizawilson', 'mzed_9', 'poketget', 'Richter_q8', 'alsumaity', 'docsrendi', 'assfess', 'fmsaa1', 'AliAlmeshaal', 'fransissscast', 'vinodlaxmikanth', 'alhayams', 'Rdec_12', 'sghiu21', 'zafiraninadia', 'xBassam', 'maryam_hasz', 'LaksanaUdin', '2LdaKyfoQQkGqku', 'M7mdAlkhamis', 'Mostafagorji2', 'Nayef_tv', 'Haniye74mhd', 'makbakul___', 'RomoKristiadipr', 'ZHALEPOTO', 'A_alkhalaf143', 'marwa_althawadi', 'R_sadatahmadi', 'Turkifa91328107', 'AliaChughtai', 'a_AsadiS', 'LaBlooshi', 'khhamid2000', 'ioixioii', 'Ro_insulation', 'maxzro3ii', 'Haji83055690', 'alanoudoct', 'manauwar_akhtar', 'davoodkarimii', 'HFR_AJL', 'AS_ALSABAH', 'msharieye', 'abu_mishriy', 'ibnseah', 'mahmoud_alaei', 'Alhasban2012', 'D7o0om_AA', 'umangabandaraus', 'Saudi_Hiking', 'UShafiq7007', 'pba4m', 'salbazie', 'Madi', 'mazin_art_', 'Ahmad_Neamah', 'abdulahaldafiri', 'waseem_rad1', 'azzawil', 'Ahmad_alqawbari', 'fahadalmajid', 'FatmaUmjannah', 'Rozelraa_', 'DrAlshoreka', 'uways_iran', 'Fah_Rat', 's_almezher', 'nasser_2030m', 'Avadhessh', 'PJdiploma', 'ayeshabintraza', 'almulla68', 'Syazwanrazzak', 'paizaneh', 'UmerElyas', 'AliJamilHaider', 'AbdulNazeeb9', 'Yaqoub_alazmi', 'Rubanahuq', 'Shyammadhan4', 'ryusaiogushi', 'NadeemZubairi', 'capt_mishra', 'PETSUITFORFUN', 'bemb_beng', 'abahiyanfikri', 'tirta_hudhi', 'fahad_alrefa3e', 'AnumLaghari', 'Sam_Gettalks', 'takiyuddin61', 'HasanAAlmulla', 'alaqsakediri', 'Nonik_k', 'piriyanew', 'ambak_', 'GusNurGL', 'YAlseddeeqi', 'hilagudnap', 'myDOA_HQ', 'drsghamdi', 'Biyan_sabil1', 'nabiel_almusawa', 'rogermend_6', 'dhayapogba', 'asalnuaisri', 'ayah_junior', 'akhtarlucknow1', 'meninakh', 'khawlamalhasawi', 'majedy011', 'Iceztiq', 'Krithika_Shukla', 'Hend_A7med', 'DrAnarkaliH', 'GuyonWaton', 'mnawaz94', 'Mraldakhil', 'IvyWamahiga', 'sanusizulkifli', 'khobar_history', 'Hamdatov', 'SurbhiChandna', 'k1k21', 'Suldaan__', 'Uppolice', 'KJimale', 'FantaMiyu', 'fi2yanoo', 'soaa1989', 'MumtazFathi', 'drcharizmaa', 'sanjaysadhwani2', 'DRHANI_ALABDALY', 'su_subit', 'yazdahoom', 'm_alsameea', 'shivesh77_kumar', 'planner_kw', 'humaid19', 'haniyeh1983', 'bu7amdan93', 'fitriazmann', 'A_AlSaif1', 'DilshadFeroz', 'thomascorbet_mo', 'mnor_alquraini', 'realupboy', 'mzinelsawari', 'monaalMaliki', 'ritika_offl', 'Muaz_Eltayeb', 'esmrutiranjan', 'appusammangi', 'skycinemas', 'naruebet_j', 'sukinameshekhis', 'AhmedJallalah', '____H7H', 'GorakhpurPost', 'drmohdizhararif', 'MumtazAICC', 'rogaia', 'kobraish', 'shahroll_SH', 'womencoalition', 'HajiMat5', 'ChahdaJalal', 'A_alomari5200', 'pailaodeh', 'ladyattorny', '3thb_90', 'anisvelvett', 'KhalifaBinM7md', 'MereMortal01', 'taimoor_989', 'V1_SA7', 'ibn_saeedan', 'FarisLangkat', 'rrr898', 'saadalmrished', 'mohamed6290', 'Reza_y32', 'kazi_91', 'Samarjournalist', 'kebby971', 'ITxDaNiPAK', 'msd3999', 'Rizwan1rizzu', 'mmadreza1375000', 'icnnara', 'MakhdumQadirPPP', 'MeJasbir', 'dyllakasmadi', 'MariamOMG', 'NasirMahmodpmln', 'QashnB', 'SamajwadiManzar', 'AAzizTalal', 'asriprahesti', 'vjkumar798', 'Nanimaheshian09', 'vaibhavkjaiswal', 'ShahmeerAzmi', 'mryamsul', 'DR_AZM', 'tahirftp44', 'eallam1', 'Kibadr', 'sajjad_official', 'Hamid_Barak', 'MSBusaidi', 'dr_m_alsubaie', 'Azizooo1', 'RaghvendrA2Y', 'SAUDI_ARMYS', 'ZeyadAbdellatif', 'Alajmi_Ammar_', 'ulvhaul', 'zaidi161', 'vrsmaiimaii', 'MyShawerma', 'JungkookUAE', 'Awadhes49745037', 'snizlyn', 'byhoon_', 'PtAshokkiradoo', 'baderalmosaieed', 'MohsinSattar_', 'iam_obriki', 'MNMPERAMBUR', 'ImTyLivingston', 'lekhachakrborty', 'ployycs', 'IlyaniFadlina1', 'SamiMaqbool', 'santhoshadv', 'AliSABKAR', 'LotbQ8', 'sandaloriginal', 'meshalmeshari79', 'HadiAlMns', 'GundiDr', 'FazleyYaakob', 'FreddieNoir_', '_9raj', 'ahmedalramadan', 'ficofachriza_', 'FidaShaikhPPP', 'yemennews10', 'RM__NaWaF', 'Esp2ni', '_sinash', 'tintinbuenaobra', 'iiMuxl', 'amoor1415', 'KaneBousso', 'enamiiir', 'jndq8', 'SMalameriii', 'wamsha92s', 'mohamed_shamsi9', 'khaledbauomy', 'lanjamaar', 'Glauberlima00', 'Asepsae08', 'Bandaralgaloud', 'khalidGO_', 'ameeque_Jamei', 'itsme_su', 'ImranMohamed25', 'aziz_hfc4', 'patis_pa', 'sdad_sdad1', 'miasanmia1400', 'lmvip', 'SaeedGhani1', 'Mittal270594', '1359fadayeiran', 'iBetho_', 'al_qassmiii', 'bn_dfna', '7asraar', 'ather1997', 'M_41x', 'kmarfaj1', 'AmmarBarlas', 'MuthusudhaD', 'KinzaShafique', 'L1HFC', 'muhammadomeriqb', 'usman77717', 'YugandharNaick1', 'pop_kamol', 'vishgoutham', 'Mohamma94019375', 'SandeepRaaaj', 'MissMalini', 'qahtam1f', 'MichaIramli', 'Alshurafa_9', 'alseid97', 'BadrBayazeed', 'HeraigeyMary', 'Jacksparrow9807', 'Brajend11406602', 'paruthiyappar', 'RumayshoCom', 'maansourrr', 'K_Mishra7', 'memyselfand_JJ', 'OTWOLheart', 'x55560', 'esamway2', 'mahmudalinurdin', 'SarahJumaa', 'tunfaisal', 'abdullahalkenan', 'wadhaalmullaa', 'COSHPSA', 'MunaHawwa', '99_barata', 'phagatleen', 'maulanadehlavi', 'Humorousunny', 'Faeez3699', 'izez73', 'Lynelyne09', 'Sheikh_Farhan4', 'msk110', 'abu_taymour', 'Tenn1969', 'mahdihgk', 'AamanTrikha', 'Sunandhaspeaks', 'mjbensal21', 'alsnid', 'zhg88', 'aldei2005', 'Seham_z5', 'MadanRavichand4', 'deema_rest', 'neelakantha', 'Thnayyan', 'Akburkz', 'mg_healthy', 'dr_A_Alomar', 'premspacefamily', 'slemanORTHO', 'kajieubay', 'imFarooqueAhmed', 'Sughrasadafdr', 'iam__surendra', 'almalek_faisal', 'HatanMortada', 'omaralomrani_', 'rm_mbahbuyut', 'iy_rajesh', 'almostashar678', 'shirdilive11', 'RoxonJo', 'ameerabbas84', 'samigodil', 'The_Math29', 'wansao12', 'himmatsinghgur1', 'SurochitaDutta', 'desichawal', 'irwancihuy', 'Chanomyen0', 'AzrilAlias', 'aziz_benj', 'Anu1Ji', 'kidnanasakid', 'VasanthyCinema', 'MdRazialampat', 'Muhairiii', 'elhendawi2', 'Jelantik5', 'SureshK89551417', 'NazimHathin', 'Nur94598278', 'aunpun_phatnari', 'malhar_pandey', 'iam_sumitjain', 'Tauheedilteja', 'mehdi_lasharii', 'iMshtag', 'catallunya96', '700Q_', 'AchsanulQosasi', 'yayagilak', 'abeinctku', 'crazyjmom', 'Khalidjassem6', 'Q8Q8_66', 'g_Alald24', 'Kimm2T', '0103Bui', 'EsVandal', 'iwahyudis', 'Adot1998', 'JPenerangan', 'ali_alzuabi', 'Ansharah_', 'GBaskar84322827', 'Krishna29892392', 'TT_Sunshine_', 'ehxstingss', 'Mugiwaramonkey2', 'M_Mddddd', 'infopkl', 'WeLoveTanon', '_h2007', 'kultsum_kh', 'ranjita_kharel', 'dianovitaap', 'urfavhijabii', 't_bin_abdullah1', 'abidMalik349', 'neyugnia', 'JKDubai1', 'nunaloey', 'AthbiAlenezii', 'SantapanMinda', 'Maymay18091', 'SSharmajsr', 'nnx700', 'yar_Dabirestani', 'geniius2', 'Hibakiyumi', 'MohdNohSalleh', 'nkalsabah', 'anjaltamang_', 'abomohamad1966', 'ainunrozi', 'iwhale', '1bentnajid', 'RhikaChandra', 'Gome_Suphawadee', 'sarvmanglamcom', 'CFhcims', 'dtnoorkhan', 'puncakmdpl', 'Dr_wisham', 'contracultural', 'FordaIndia', 'EachStepAhead', 'itsactuallynazz', 'bambam_yani', 'artemis_lady', 'staronline', 'gauravtiwarirau', 'sanggovsa', '2GetherSeries', 'alhamdawii', 'vqueeram', 'loveinPPE', 'RanjanaS012', 'mahasiswaYUJIEM', 'realmanoj_111', 'alibest55', 'ChinVij', 'Lovely_Ranee', 'Puthutea', 'FoxKid1302', 'GoodDayinMay', 'suwaidi_jamal', '3bood2210', 'Bodh_123', 'te1_sa', 'veraAvellina', 'l60v_', 'nattapolc', '_NadeemZaidi', 'air_selangor', 'Sherisa_CJ', 'Rizalar84687565', 'DJRayLais', 'Ntdsasi', 'ras_iben39', 'I_am_deval', 'siapaygtw', 'habakr', 'mirzaiqbal80', 'pfcnarak', 'ShuhadahAzman', 'hazwanmohamad', 'nobucoltd', 'hksharma678', 'doodlybish', 'samarjavaid', 'SaintZeeSup', 'KSA24', 'ExoLMahsa_1', 'DaryonoBMKG', 'NAofPakistan', 'SyedMusaGillani', 'AlAnba_News_KW', 'GuruDharanidhar', 'dnbgsp', 'Eyaaaad', 'wtfaishah', 'MKupperman', 'snodgrass_jo', 'PE_ResearchUnit', 'Alsaud72', 'SoesoeKoetjink', 'Jackhangu', 'RIYADH_MUTAHHIR', 'akyllus', 'josemi422', 'bigfoo1119', 'Caporal_abdel2', 'fahmirusliMFR', 'T_Nenninger', 'vietnamenglish', 'farhanko888', 'bo3dla5', 'spkatidrid', 'Bin_QoMaNi', 'JournalistJagd2', 'Pareshb83781485', 'pandey_shukdeo', 'afilliee', 'lulu_chan92', 'rezaramatf', 'ChidiNwatu', 'RehamKhan1', 'ZandyMAAQ_01', 'ViWiWrites', 'raasa_cokelat', 'shakmademecrazy', 'ramah_adawi', 'jjjehjong', 'AlaqeelMe', 'K_ErichanJKT48', 'Falcon_kuw', 'almadeedd', 'tedmaldives', 'mocha_diano', 'YouFreemen', 'Dinda20981194', 'rsrikanth05', 'Harendr56135574', 'Sarah_AlSayegh', 'juanyaxo', 'arifinpribadi', 'chaerulpunya', 'IbrahimDashti_', 'Eig_Banphot', 'adriansyahyasin', 'kasemsanmeethip', 'fazliwazil_', 'MACzilla_', 'mpmanishyadav', 'Hyo_Joestar', 'TeeKhaled', 'mub_alotaibi', 'NewEarthquake', 'Alabdulgaderaa', 'TMCPoldaMetro', 'sanjibroy360', 'salahalzamil', 'naifibrahim1', 'FaridaAlHabib1', 'sarahalmulhem', 'sahel_almakha', 'fadwni', 'gajanan137', 'shafeekayousif', 'ahmed_narysh', 'CharanSingh___', 'alkhawash', 'rameshwar4111', 'hamodialrubesh', 'tareemed2002', 'AyushiS96622915', 'mbsamari', 'KakarFoundation', 'Thehulkey', 'itsafifyalll', 'MrMan9or', 'RizwanPehelwan', 'MAl7ardan', 'atulpradhansp', 'oi1_2', 'Kemmooalharbi', 'Naszaman59', 's_najafi57', 'Eidalsawy222', 'iamrudra808', 'WeAreBandook', 'lalshareef', 'pong32', 'imKP_Indian', 'Mahdi__Ahmadian', 'Khalid_Shukaili', 'pistachio__app', 'IMohdBH', 'aMrazing', 'panathurjyothe1', 'c7uui', 'CoachJustinL', 'Skyriswan', 'Leechansew14', 'arehmanbutt', 'realm2110', 'lLxbbit', 'astoldbyashia', 'heybrian__', 'CmaRakeshSingh', 'thanyas13', 'noiseandgrains', 'keathamreddy', 'hamed_pahlevani', 'chanya_nath', 'Kinnzayyy', 'Nurii____', 'azizooghali', 'Nawafalbugmi', 'belakoboli', 'Unum4', 'sami_1photo', 'FALAH_AL_7THLEN', 'Musayeb_1', 'Javedkhan3245', 'blqsdzkrh', 'bangmuehhh', 'aishaalabdullah', 'Hamebrahimi', 'TalalAlTannak', 'Mohsen2015_', 'mosabh95', 'sultanaalsuda', 'celiaasari', 'chumsrising', 'naziarubbani', 'rezashamsa', 'Sana_26Feb', 'ik_tiger_', 'iki_inabdalla', 'loveshehnaaz12', 'F6ammyh', 'mdsjaihind', 'khalifa_alhinai', 'Shalhoub88', 'this_xaimah', 'hsaqib17', 'Jahazi_Bhaiya', 'akshaynatu', 'everfilm_', 'Mathen_Naidu', 'lawyers_society', 'munirharithzz_', 'abmsdyab5136', 'FarhanAries007', 'mohad_hinai', 'gulistan2009', 'areejun_', 'aishamdidi', 'RidhwanRafiee_', 'anabreco', 'AShukkla', 'Sidhearts_unity', 'JorSPSS', 'ashishMib', 'Adhek_Byt', 'Tweety06731326', 'Sudhi_Manchira', 'KKESHKSA', 'DehradunSsp', 'spywalker', 'aditiraohydari', 'TheRashamiDesai', 'humanwithpr0b', 'Pi_Pooh', 'alhussainiOS', 'mkq_5', 'PatrickMoodyFCO', 'Commonman_1100', 'AkmalRajjput', 'chinnineni72', 'guru4073T', 'MeeaadAljowder', 'Weyakqa', 'unilubis', 'Mohag24', 'SHE_Maldives', 'ArchitectsG', '1ALShathri', 'Vismrita2007', 'haroon_natamam', 'MuzzPlhiv', 'pamiipams', 'fiezanizmohamed', 'sheggario', 'amiryaghoubali', 'MohamadOlia', 'Roadcare_Msia', 'MianMohsinFSD', 'Araya75218557', 'm9rrm', 'sdousary', 'seanfreestyle', 'narsfdll', 'Aliaaljraiwi', 'kah1154', 'itsthepianoguy', '9yaud', 'dr_almalky', 'AljaidahJ', 'RuggedyBaba', 'khalilalbalush1', 'hatem_hus74', 'Sauddaghistani', 'adelalawadi', 'FAISAL_1410', '5lifabinK', 'mrmraam', 'SVLVM', 'tofutoomis', 'mxmoonn_', 'satish_sekhar', 'QasimFaraz', 'AboHanen1907', 'naif_altiman', 'aboriyadh_5544', 'Sh_alnasser', 'abSayyar', 'EngrIftikharCh2', 'HiroBeing', 'DrvijaySwami', 'Omar__Cule', 'Kalmhrzi', 'HarfCafe', 'falhajry75', 'manojkumarmukul', 'HaniRajab', 'jnexx_', 'King_of_Asgaard', 'rajpuro10', 'mdhaikalariffin', 'archie_hs', 'Anamika7018', 'alhumaidealharb', 'adzman86', 'drmostafa64', '_mohammedmahadi', 'jebat_165', 'NaanumEngineer', 'KanumuriVikas', 'Surawuti', 'Krishna01525', 'rahla22', 'Amaresh03335514', 'xrockybrux', 'alyahyan', 'yukitsumu21', 'soeyoto1', 'kundan05294099', '8F9_0', 'AlShammari62', 'lerenofa', 'agapol123', 'halameer78', 'm3moool2', 'lubrifica_mos', '_abdalux', 'didbaniran', 'SeeWithSound', 'uae_1_71', '7amod_seko', 'kokumaro_1969', 'a_aldhahouri', 'BoopalanVJ', 'thatnorow', 'DrNasseralajmi1', 'ArifBelg', 'alshehri_dr', 'IsharaDanasekar', 'Almuhaml', 'chloekhanxxx', 'dr_yousif_m', 'AlRefai_T', 'MukhtarNuur', 'MohamedCambe', 'Sagar___Ram', 'RupantaraD', 'moonchild62ali', 'sandyyy_20', 'sharadasrinidhi', 'Setivijay', 'bestone1404', 'Ekbyte', '2abd13', 'krunalMore6', 'KKYOfficial', 'puengneepai', 'LatifaSY_', 'Yahya_Busaidi', 'riyad1000', 'KhaledBnFaisal', 'ayudwahyuni_', 'Abhishek4_BJP', 'JasMatta', 'suryasuresh424', 'mnmalmulhim2012', 'APillania', 'kadibatarka', 'BabaBlocked', 'AttiqurSatti', 'obaid1970', 'nwaf201000', 'hasanasadiz', 'Sinyar_om', 'S__Allah', 'NMuthukrishnan0', 'jekajakel', 'SadeemKhamees', 'nalni_gupta', 'osamajeid', 'zuhairimisrawi', 'StillBreath90', 'pax_pack', '_moalmosawi', 'SalemmAldawsari', 'Maybe3li', 'droalh', 'fatimanoor97', 'udafajri', 'azamgrhbabaG', 'fardinahda', 'zakishah82', 'thahitun', 'msferab88', '6YwLa6oy80pB85S', 'CJoelLumanglas', 'sanaelshammri', 'WutMovie', '_ibrahiim', 'Zuwenasalim', 'mohammadsamieii', 'banderalrezohan', 'deepaksingh83', 'uae_052', 'azizalablan', 'dahem_aldosari', 'aryww33', 'joule_a0', 'omarsport', 'jamiamillia1', 'gxbbyxz', 'HaleemAdil', 'mnormanhakim_', 'oubiaofeng', 'AjayBrahman', 'InuuMaajid', 'najaf72_', 'MessUniverse', 'Shahrukh_PPP', 'Fauzanjadid', 'MeriTishnagi', 'Ali_alk3bii', '8pisit_', 'a7medo__5', 'puterysolehah_', 'sarchana1016', 'Real_pawan1', 'lazarooni', 'ShabahatAShah', 'nawafzrr', 'SurryiaN', 'ManelKape', 'yogitabhayana', 'abu_nasser300', 'misz0076', 'SiddharthNews18', 'TariqueHussainQ', 'mutasem_mamari', 'DrZ_84', 'Ams_qatar', '3bduullaa', 'aka2139', 'fachhusa', 'LisaAmartatara3', 'Awf_Alshehhi', 'MarikaRahman_', 'papakrab', 'tubagussalim', 'naseralsudairi', 'ArdyantoYudi', 'bangarip_', 'salahalsamhan', 'omvirlather', 'Indiansontop', 'JohnyBravo4444', 'poeticpicture', 'PSKSMPI', 'Moenthelicious', 'SukimanMerapi', 'Faldoughji', 'G4A_News', 'kurniawanuzy18', 'nicolesamudio', 'DrJasimKhalfan', 'DianJamil', 'Ceaser14', 'fozisidd', 'Greschinov', 'RafedAlHarthi', 'Ashril_92', 'shiva_U1ian', 'HananAlkhawaja_', 'MariamSulieman', 'Adel_AlMishry', 'mohsaud08', 'rowalghofaily', 'Maher_Alarnaoot', 'a_alwthah', 'waliuddin_Al', 'HawraHabeeb', 'AlanoudAlhnaki', 'mubarak_harbi', 'Babulal16495564', 'tanmay_shinde99', 'Noufshq', 'Praveen_Gkp20', 'asyrafdeeno1', 'LingLearm', 'ibijalpatel', 'sohar943', 'manojsk5429', 'Mohammed_Lori', 'binmeizar', 'PandeyAjay_IND', 'slmesaf', 'lovcoffe', 'Azzabinsulaiman', 'G_s_Prakasam', 'AmirulSyafiiqqq', 'Mohd_Salman0020', 'TriArsala', 'MathildeLAzou', 'navalkant', 'GlobalVillageAE', 'hforhammadd', 'asro_12', 'abdullahaserey', 'justscorpion', 'EssamAlGobaisi', 'iimoha_9', 'DenyakDenyay', 'alahsekaer', 'peerapa77128989', 'alyousef28', 'Mariam_Yagoob', 'zayedAB', 'neenakulkarni', 'shaheen4_', 'EjazGujjar55', 'hxuz6xrAm2FQQTo', 'bn_whga', 'PHO_AhmedShehab', 'rezinejabnet', 'NuhaAlHazzaa', 'Meth_Pramudi', 'AbirAbdulRahim', 'rabimishra', '1mcStylish', 'm77_alshehri', 'PratamaMario', 'TasneemJafri1', '7amadalyafei', 'liaaaaa33__', 'affandysfeby', 'Renreno', 'Draldajani', 'hariom_8055', 'salm966', 'derbas82', 'derradjihafid', 'Shiphattey', 'fordder', 'alsheedi94', 'ibrahim70alahmd', 'ekb_9', 'khd_uae', 'whatchelsysays', 'ASalwe', 'hafizrizaal', 'ali__alison', 'Rdcci_Trc', 'the_tea_lover_', 'Moh1Rz2H3', 'alshiadi999', 'JaiHMehta', 'aomaomii1', 'AbdulazizAlatt1', 'DC_physicist_YQ', 'DhofarMun', 'alseiari01', 'MoSamUA07', 'ahmed7k', 'mishravishal', 'dhagandey', 'Salibh1', 'sudhirs03792226', 'CoptertTia', 'jalalsherazi', 'cmcamel', 'rashed_alajmi24', 'mythreyaa', 'MorvaridSaied', 'NANDLALMAURYA', 'AlsaneaAhmed', 'SathishAneNenu', 'Ravi_kashyap1', 'Brijlalipsbjp', 'Ashokku47487877', 'kohistani_nawaz', 'MyNameIs_SAGAR', 'alia1994alia', 'adit_insomnia', 'djfarezki', 'AnasZahrin', 'RaviJanghela', 'Siddharthavat12', 'alfiantors', 'DoelRobertz', 'saifuddeen89', 'dkn25072738', 'Niru_Dcruz', 'ohtweet', 'gzgahtani', 'aboo_eshagh', 'smhqa', 'Mohitmonty21', 'hamedalajlan', 'MostafaGhalava4', 'SyedSameershah0', '900Delta', 'ahmedaljad90', 'HappyMeal05_', 'vlaauthor', 'kahilan9999', 'yousef_aldakhel', 'TourismSo', 'akkii_jp', 'Aman_aich_', 'saibandreddi', 'TeamEXOIndia', 'latttiiifffaaa', 'info_bank_', 'anmolpreetkaurr', 'Abdulaziz_G61', 'hilalalsarmi', 'leena_amiri', 'isunnyshah', 'M7dmarar', 'ZahidLashari15', 'AdityaSingh_82', 'resyntand', 'HamedMerah', 'MoBrg_', 'zuhairalsiyabii', 'AlKoohejiAdel', 'nitinbhatnagaar', 'BunnyKarthik77', 'AdmanojM', 'gole_yass', 'XavierMariabell', 'ramzul_alam', 'aray_shiva', 'nehakum79798495', 'Reikun2010', 'qa6an', 'CrickeTendulkar', 'CHOTIWALA', 'snigdhaman', 'atengsepet', 'almaktutat', 'nishantbhuse', 'Torathuna', 'om1live', 'putraditama', 'KeshvalaJay', 'gauravstvnews', 'sharmarekha', 'SubbarajuSiva', 'julianamaulida8', 'Ali_Alsultan0', 'UmarKamani', 'PASSI0N___', 'kurniaasp', 'Mohammad8ahtani', 'pisces5401', 'husnakuzer', 'masfaizhakim', 'MayangSchreiber', 'sarhaniy', 'fahadqtr3', 'jogokariyan', 'S_AISheikh', 'ABJ506', 'Jiraporn2363', 'fahiraidris', 'Eltnhed', 'Alyazi_uae', 'AhmAbdelKawy', 'fjb223_', '7almohannadi73', 'vip5402', 'Bratzaat', 'Hq97f', 'hussaindeabel', 'aazizalnasr', 'ankuragar01', 'arifksyed', 'shantprakash', 'Arsiiimughal', 'aslamismaill', 'ItzTayyaba_', 'adeebihsann', 'Indori_bewda', 'majdialsubayhi', 'baderlq8', 'ped_orl', 'ghassan1980', 'JanbazSMBB', 'Real_Hadaji', 'tareqalmuhawesh', 'h_aburatani', 'xmoh72', 'GeraldineHannaa', 'king_fahd7', 'Pardeepkaushik1', 'saidx731', 'Amxrull', 'IndianMaratha', 'TCGEHLOT', 'M3Almuhairi', 'Archana_Sunil_', 'mindmelby']] of <class 'list'>

In [106]:
type(user_name_list)

list

In [107]:
user_name_list[:20]

['Clinical_Sim',
 'AlexInAir',
 'KVishReddy',
 'one_by_two',
 'JebiMather',
 'NNN_TNP',
 'prasarbharati',
 'YohanesPatty2',
 'thehaelyn',
 'SAFE_JOHNY',
 'srijanpalsingh',
 'Zayed_U',
 'manateedugong',
 'mbah_mijan',
 'ShaikhHamzaIMS',
 'IamOnir',
 'RiaRevealed',
 'CPEC_CSCEC',
 'dbeasley1',
 'svirtuoso2010']

In [110]:
len(user_name_list)

4631

We collected over 10,000 tweets from 4631 users. Tweets are mainly about corona we could also collect tweets related to any other topics, we can also specify different hashtags, locations, retweet counts etc.

These tweets are saved in csv file "tweets.csv"

As we can see, we could only mine the tweets and information related to a particular tweet, we also need additional information regarding the user. To mine the user info, we use the screen name of the twitter user. User info is generated in #2User_info_scraper.ipynb 